\textbf{Genes are elongated by TE insertions and TEs contain information on gene expression levels}
\textbf{(A)} Distribution of gene length for various well annotated species. Red lines indicate means and whiskers indicate 1.5*IQR (inter quartile range). Blue bars are all protein coding genes and yellow bars are the subset of genes with homologs in all species. (human:Homo sapiens; chimp:Pan troglodytes; monkey:Macaca mulatta, mouse:Mus musculus; dog:Canis lupus familiaris; chicken:Gallus gallus; frog:Xenopus tropicalis; zebrafish:Danio rerio; fly:Drosophila melanogaster; worm:Caenorhabditis  elegans)
\textbf{(B)} Histograms of segments inserted into the human genome compared to chimp (left) and mouse genome compared to rat (right) by length (See methods). Peaks near 300bp corresponds to Alu, and near 6000bp corresponds to LINE. Pie charts (insets) indicate fraction of inserted bp overlapping transposable elements (TE) and other types of repeats.
\textbf{(C)} Percentage of ATAC peaks overlapping repeats. Left side: for all ATAC peaks, right side: for ATAC peaks overlapping recently inserted segments calculated in (A). 
\textbf{(D)} Schema describing repeat score and regression model. Repeat scores (upper panel) are calculated separately for each type of element and for each gene as the count in the specified interval. Regressions (lower panel) are calculated separately for each cell type by fitting coefficients (X) to ranked expression levels (Y) using intercept(a) and coefficients (b) selected to minimize mean squared error. 
\textbf{(E)} Fits to 80\% of the genes are cross validated using the remaining 20\%. Histograms show $R^2$ for each cell type (blue), and for controls shuffling repeat scores for genes (green) or changing the repeat score by randomly changing the location of repeats (red) or by calculating the repeat score over a randomly selected genomic interval of the same length as the gene (orange). The latter two shuffling methods  retain some correlation relative to gene shuffling (green) since they maintain the correlation between gene length and expression (See Supp.Fig.XX).
\textbf{(F)} A model of how neuronal genes become elongated over evolutionary time scales. 
