% Abstract < 150 words 
\begin{abstract} 
Mammalian neuronal diversity is well documented, but the principles underlying this diversity are poorly understood. To begin to assess brain-wide transcriptional diversity we obtained RNA-seq data from 179 genetically identified neuronal classes from diverse brain regions.%assess how the diverse neuronal cell types are realized, we prepared RNA-seq data from 193 genetically identified neuronal classes.\%
Using improved expression metrics that distinguish information content from signal-to-noise-ratio, we found that homeobox transcription factors contain the highest information about cell types and have the lowest noise. Whereas, effector genes contributing most to neuronal diversity tend to be long, due to long introns, and are enriched for synaptic genes. Genome accessibility measurements reveal that long genes have more candidate regulatory elements arrayed in more distinct patterns. These candidate regulatory elements frequently overlap interspersed repeats and the pattern of repeats is predictive of gene expression. Elongation of neuronal genes by insertions of mobile elements and the resulting new regulatory sites may be an evolutionary force enhancing nervous system complexity.

%% currently 141 words 
\end{abstract}
