% Abstract < 150 words 
\begin{abstract} 
The transcriptional diversity of mammalian neuronal cell types is well documented, but the principles underlying this diversity are poorly understood. Here we present RNA-seq data from 189 genetically identified cell types. To quantify expression diversity we introduce two metrics that distinguish information content from signal-to-noise-ratio. Homeobox transcription factors, as a family, contain the highest information about cell types and have unusually low noise. Highly informative effector genes tend to be long. Long genes are enriched for synaptic genes, are repressed in non-neuronal cells, and contribute disproportionately to neuronal diversity. Consistent with having more diverse neuronal regulation, genome accessibility measurements reveal that long genes have more candidate regulatory elements arrayed in more distinct patterns across a sample of neuronal cell types. This may have implications for the evolution of complex nervous systems, as species with more complex brains have longer introns in orthologous neuronal genes presumably via insertion of transposable elements.

% currently 152 words 
\end{abstract}
