% Abstract < 150 words 
\begin{abstract} 
Mammalian neuronal diversity is well documented, but the principles underlying this diversity are poorly understood. Here we present RNA-seq data from 193 genetically identified cell types. To better quantify expression diversity we distinguish information content from signal-to-noise-ratio. We find that homeobox transcription factors contain the highest information about cell types and have the lowest noise. Effector genes contributing most to neuronal diversity tend to be long, due to long introns, and are enriched for synaptic genes. Consistent with needing more diverse regulation, genome accessibility measurements reveal that long genes have more candidate regulatory elements arrayed in more distinct patterns. Candidate regulatory elements frequently overlap interspersed repeats and the pattern of repeats is predictive of gene expression. This may have implications for the evolution of complex nervous systems, as species with more complex brains have longer introns in orthologous neuronal genes presumably via insertion of mobile elements that evolve into cell type-specific regulators.

%% currently 152 words 
\end{abstract}
