% Abstract < 150 words 
\begin{abstract} 
The transcriptional diversity of mammalian neuronal cell types is well documented, but the principles underlying this diversity are poorly understood. Here we present RNA-seq data from 189 genetically identified cell types. To quantify expression diversity we introduced two metrics that distinguish information content from signal-to-noise ratio. Genes with lowest noise are enriched in homeobox transcription factors. As a family, they contain the highest information on cell types. Genes with high information content are biased towards longer genes. Long genes are enriched for synaptic genes and specifically silenced by REST in non-neuronal cells. Thus, neuronal diversity is mainly realized by long genes. Consistent with this view, genome accessibility measurements reveal that long genes have more candidate regulatory elements and these occur in a larger number of distinct patterns across a sample of neuronal cell types. This may have implications for the evolution of complex nervous systems, as species with more complex brains have longer introns in orthologous neuronal genes by insertion of transposable elements.

% currently 162 words 
\end{abstract}
