% Abstract < 150 words 
\begin{abstract} 
The mammalian nervous system is constructed of many neuronal cell types, but the principles underlying this diversity are poorly understood. To begin to assess brain-wide transcriptional diversity, we sequenced the transcriptomes of the largest collection of genetically and/or anatomically identified neuronal classes from throughout the central nervous system. Using improved expression metrics that distinguish information content from signal-to-noise-ratio, we found that homeobox transcription factors contain the highest information about cell types and have the lowest noise. Non-transcription factors that contribute the most to neuronal diversity tend to be long, due to large introns, and are enriched in genes specifically involved in neuronal function. Genome accessibility measurements reveal that long genes have more candidate regulatory elements arrayed in more distinct patterns. These candidate regulatory elements frequently overlap interspersed repeats and the pattern of repeats is predictive of gene expression. Elongation of neuronal genes by insertions of mobile elements and the resulting new regulatory sites may be an evolutionary force enhancing nervous system complexity.

%% currently 150 words 
\end{abstract}
