% Abstract < 150 words 
\begin{abstract} 
Abstract version 1

The transcriptional diversity of mammalian neuronal cell types is well documented, but the principles underlying this diversity are poorly understood. We present a new resource of 2.2 Tbp of RNA-seq data from 189 genetically identified cell types, obtained using methods designed to maximize accuracy and precision, relative to those obtained from single cells. Simple new metrics of the information content of genes about cell types, and the noisiness of differential expression reveal that homeobox transcription factors have uniquely low-noise expression and maximal information about cell types. Highly informative effector genes are enriched in long genes. Increased gene length may provide the increased opportunities for gene regulation needed to achieve increased neuronal diversity. Consistent with this view, genome accessibility measurements reveal that long genes have more candidate regulatory elements and these occur in a larger number of distinct patterns across a sample of neuronal cell types.

Abstract version2

The transcriptional diversity of mammalian neuronal cell types is well documented, but the principles underlying this diversity are poorly understood. We present a new resource of RNA-seq data from 189 genetically identified cell types. To quantify expression diversity we introduced two metrics: differentiation index (DI) and signal contrast (SC). DI quantifies the amount of differential expression and information content. SC quantifies how robust the differential expression is. Genes with high SC's are enriched in homeobox transcription factors. As a family, it contains the highest information on cell-types than other families, consistent with the idea that they are terminal selectors. Genes with high DI's are enriched in long genes, which are characterized by synaptic genes and not expressed in non-neuronal cells. Thus, the diversity of neuronal transcriptomes is mainly realized by long genes. This may have an implication in the evolution of complex nervous systems.

Abstract version3

The transcriptional diversity of mammalian neuronal cell types is well documented, but the principles underlying this diversity are poorly understood. Here we present RNA-seq data from 189 genetically identified cell types. To quantify expression diversity we introduced two metrics that distinguish information content from signal-to-noise ratio. Genes with lowest noise are enriched in homeobox transcription factors. As a family, they contain the highest information on cell types. Genes with high information content are biased towards longer genes. Long genes are enriched for synaptic genes and specifically silenced by REST in non-neuronal cells. Thus, neuronal diversity is mainly realized by long genes. Consistent with this view, genome accessibility measurements reveal that long genes have more candidate regulatory elements and these occur in a larger number of distinct patterns across a sample of neuronal cell types. This may have implications for the evolution of complex nervous systems, as species with more complex brains have longer introns in orthologous neuronal genes by insertion of transposable elements.

% currently 162 words 
\end{abstract}
