% Abstract < 150 words (elife)
\begin{abstract} 
The principles governing neuronal diversity remains a fundamental goal for neuroscience. By separately analyzing the pattern and robustness of expression diversity across nearly 200 genetically identified cell types, we identify two contributing gene classes that differ in their physical properties. Short homeobox transcription factors comprise a combinatorial code distinguishing cell types and exhibit low transcriptional noise likely reflecting inaccessible chromatin. Long effector genes, such as channels and cell adhesion molecules, contribute disproportionately to neuronal diversity. Genome comparisons reveal mobile element insertions are excluded from short, low-noise genes, but largely account for elongation of neuronal effector genes. Chromatin of long genes contain more candidate regulatory regions, arrayed in more complex patterns, and often overlap mobile element insertions. Conversely, patterns of mobile elements are predictive of gene expression patterns. We propose that elongation of genes by mobile elements insertions is a genetic platform for increasing neuronal diversity over evolutionary time scales.
%Describing the cell types of the brain and the principles governing their diversity are fundamental goals for neuroscience. By comparing the transcriptomes of nearly 200 genetically identified cell types, we identified two gene classes that underlie distinct aspects of neuronal diversity. We find that short homeobox transcription factors exhibit extremely low transcriptional noise likely due to their less accessible chromatin as revealed by ATAC-seq. They also form a combinatorial code to distinguish cell types, suggesting a possible role in driving cell type identity. In contrast, long genes, which are mostly effector genes such as channels and cell adhesion molecules, disproportionately contribute to the information content of neuronal diversity. From ATAC-seq, we find that these long genes contain a larger number of candidate regulatory regions arrayed in more complex patterns, suggesting increased capacity to be differentially regulated. Many of these putative regulatory regions are found in regions that contain mobile element insertions. Conversely, patterns of mobile elements distribution contain information of gene expression patterns. These mobile elements are also largely responsible for the increased length of long genes. Different species contain different sets of mobile elements and the extent of gene elongation are different across species, human having the longest set of genes and the length of gene mostly correlated with the brain complexity. We propose that elongation of genes by mobile elements insertions is a genetic platform for increasing neuronal diversity.

%Insertions of mobile elements have progressively enlarged vertebrate genomes and massively elongated homologous genes. Long genes are preferentially expressed in the brain but the reason for this is unknown, and is surprising, since expression of long genes comes with significant metabolic cost and increased susceptibility to mutations causing cancer and neuropsychiatric diseases. Here, by examining gene expression in nearly two hundred genetically identified cell types in the mammalian brain, we show that long genes contribute disproportionately to neuronal diversity. Assays of genome accessibility reveal that candidate regulatory elements occur in greater numbers and in more diverse patterns within the introns of long genes and these overlap inserted mobile elements. The complement of mobile elements present within or near genes is predictive of relative expression levels. Hence, by increasing regulatory complexity, gene elongation by mobile elements may facilitate the emergence of more complex brains comprised of more diverse cell types.
\end{abstract}

%The mammalian nervous system is constructed of many cell types, but the principles underlying this diversity are poorly understood. To assess brain-wide transcriptional diversity, we sequenced the transcriptomes of the largest collection of genetically and anatomically identified neuronal classes. Using improved expression metrics that distinguish information content from signal-to-noise-ratio, we found that homeobox transcription factors contain the highest information about cell types and have the lowest noise. Genes that contribute the most to neuronal diversity tend to be long and enriched in factors specifically involved in neuronal function. Genome accessibility measurements reveal that long genes have more candidate regulatory elements arrayed in more distinct patterns. These elements frequently overlap interspersed repeats (mobile elements) and the pattern of repeats is predictive of gene expression. New regulatory sites resulting from elongation of neuronal genes by mobile elements may be an evolutionary force enhancing nervous system complexity.
%% currently 144 words 


%The mammalian nervous system is constructed of many cell types, but the principles underlying this diversity are poorly understood. To assess brain-wide transcriptional diversity, we sequenced the largest collection of genetically/anatomically identified neuronal classes. Using improved expression metrics that distinguish information content from signal-to-noise-ratio, we found that expression of homeobox transcription factors is tightly controlled and their binary expression pattern can distinguish 98\% of cell types profiled. Genes that contribute the most to neuronal diversity tend to be long and enriched in factors specifically involved in neuronal function. Genome accessibility measurements reveal that long genes have more candidate regulatory elements with distinct patterns. These elements frequently overlap interspersed repeats (mobile elements) and the pattern of repeats is predictive of gene expression. This suggests that regulatory potential of long genes have been enhanced by mobile element insertion. Thus, elongation of neuronal genes by mobile elements may be an evolutionary force to expand brain complexity.


