
\section{Discussion}

\subsection{A Resource of Genetically Identified Neuronal Transcriptomes}

The dataset presented here is the largest collection of cell type-specific neuronal transcriptomes (Table 1). The approach employed in this study provides a complimentary view of neuronal diversity to that afforded by single cell sequencing. By sorting and pooling $\sim$100 cells chosen based on genetic and anatomical similarity, we generated profiles with low noise and high depth, but, where tested, with a comparable degree of homogeneity, as that obtained in recent single cell studies. 

The fact that each transcriptome corresponds to a genetically (or retrogradely) labeled population will foster reproducible studies across investigators. The few profiles in our study that mapped to more than one single cell profile (Figure 1 Supplement 3), could represent cell types better distinguishable using single cells or improved genetic markers. However, in most cases, the single cell clusters were barely separable (Figure 1 Supplement 9), and the two available single cell studies did not agree. Given the complimentary advantages of improved reproducibility and separability afforded by our approach, and of reduced heterogeneity afforded by the single cell approach, further integration of these methods with other modalities, such as FISH \citep{Moffitt_2016} are needed to accurately profile the full census of brain cell types. The present dataset is a major resource in these efforts. 

\subsection{A transcriptional code for neuronal diversity}
We utilized easily calculated metrics that capture essential features of the robustness and information content of transcriptome diversity. These measures are not cleanly captured by variance-based metrics like ANOVA and CV (Figure 2 Supplement 1). We found that the homeobox family of TFs exhibited the most robust (high SC) expression differences across cell types (Figure 2C). These ON/OFF differences were characterized by extremely low expression in the OFF state (Figure 3A-D). Mechanistically, the low expression was associated with reduced genome accessibility measured by ATAC-seq (Figure 3C,D), presumably reflecting epigenetic regulation of the OFF state, known to occur for example at the clustered Hox genes via Polycomb group (PcG) proteins \citep{Montavon_2014}. Although this regulation has been studied most extensively at Hox genes, genome-wide ChIP studies reveal that PcG proteins are bound to over 100 homeobox TFs in ES cells \citep{Boyer_2006}. Our results indicate that strong cell type-specific repression persists in the adult brain, presumably due to the continued functional importance of preventing even partial activation of inappropriate programs of neuronal identity. 

Although individually, homeobox TFs contain less information about cell types than long neuronal effector genes, their patterns of expression are highly orthogonal and therefore their joint expression pattern is highly informative. As a group, homeobox TFs distinguished 98\% of neuronal cell types profiled. (Note this includes several Purkinje and Hippocampal pyramidal cell groups that may actually represent duplicate examples of the same cell types). Historically, homeobox TFs are well known to combinatorially regulate neuronal identity in Drosophila and C. elegans \citep{Kratsios_2017} and the vertebrate brainstem and spinal cord \citep{Dasen_2009,Philippidou_2013}. Our results suggest a broader importance of homeobox TFs throughout the mammalian nervous system. Continued expression of these factors in adult neurons suggest they likely also contribute to the maintenance of neuronal identity.

%The continued expression of homeobox TFs throughout the adult mammalian nervous system suggests that they likely also contribute to the maintenance of neuronal identity.
%Our results suggest broader importance of homeobox TFs throughout the mammalian nervous system. Continued expression of these factors in adult neurons suggests they likely also contribute to the maintenance of neuronal identity.

%In order to reveal the relationship between specific cell types and TFs, we constructed a TF decision tree for classifying profiled cell types. As expected from their high information content, homeobox TFs figured prominently in this list (49/127). Many of the identified factors are known to be key transcriptional regulators of the cell types in which they continue to be expressed (Supplemental Table 3). In most cases it is not known whether or not these roles occur only in development, or are also important for the maintenance of neuronal identity. Lists of expressed TFs and the genetically accessible cell types in which they are expressed provide a ready source of testable hypotheses about how cell type specific transcriptional identity is maintained in the adult nervous system.

\subsection{Long genes shape neuronal diversity}
Our study suggests that long genes contribute disproportionately to neuronal diversity (Figure 4A-C). Increases in the number of alternative start and splice sites present in longer genes may contribute, but in addition, we hypothesize that longer genes have a larger number of regulatory elements that alter expression and enhance differential usage of these alternative sites which is most beneficial in the brain. Long genes likely elongate during evolution, via insertions of mobile elements in their introns \citep{Sela_2007,Grishkevich_2014}. Long neuronal genes, such as ion channels and cell adhesion molecules, may be expressed primarily late in development \citep{Okaty_2009}. Later and more restricted expression may make these parts of mammalian genomes more tolerant to mutations caused by insertion of mobile elements. Also, mobile element insertions occurring randomly are expected to happen more frequently in long genes, thereby accelerating their elongation over the course of vertebrate evolution. In addition, insertion into long genes is likely favored (Figure 5 Supplement 1C) since insertions into exons is more likely to disrupt gene function and be selected against and long genes have a much small fraction of their length devoted to exons (Figure 4 Supplement 1A) %This trend may contribute to the massive increase in repetitive sequences present in primate genomes relative to rodent genomes \cite{Chuong_2016}. \%

Conversely, genes such as Hox genes, which are critical for early development, and often expressed in progenitors giving rise to many cell types, are remarkably transposon impoverished \citep{Chinwalla_2002,Simons_2005}. This may reflect selection against insertion but may also reflect chromatin that is non-permissive for insertion in germ cells and the early embryo where heritable transposition is most likely to occur. The high SC/low noise of many of these genes detected here may reflect a transcriptional signature of this class of genes. Highly restrictive chromatin at these genes may be established early in development to protect them from disruptive transposition \citep{Montavon_2014}, but also appears to be maintained in postmitotic neurons, thereby preventing transcriptional signals associated with inappropriate neural identities. 

Retroelement insertions have been reported to generate new promoters, exons and distal regulatory elements through a process called "exaptation" (for review see \cite{Chuong_2016}). For example, the set of neuronal genes regulated by REST has been suggested to have evolved through retrotransposon-mediated duplication of REST binding sites \cite{Johnson_2006}. Similarly, mobile elements have helped distribute enhancers important for mammalian innate immunity \cite{Chuong_2016a}, shaped genome regulatory organization via distribution of the vertebrate insulator component CTCF \cite{Schmidt_2012} and helped diversify placental function across mammalian lineages \cite{Chuong_2013}. These genetic innovations were originally seen as fortuitous benefits of an evolutionary arms race between the "selfish DNA" of mobile elements and the mechanisms by which host genomes suppress them. These mechanisms include the Kruppel-associated box (KRAB) domain-containing zinc-finger proteins, which are hugely expanded in mammals, especially in primates, where the evolutionary arms race has been explicitly documented \cite{Jacobs_2014}. Intriguingly, recent studies of these proteins indicate positive selection persisting long after the TEs they evolved to suppress lost their ability to transpose, suggesting that the arms race itself has been co-opted for other regulatory purposes \cite{Imbeault_2017}. 

Here we provide evidence supporting the hypothesis that evolution of the vertebrate nervous system may have taken advantage of mobile element insertions and subsequent exaptations to diversify neuronal cell types, increasing the complexity of brain circuits. Long genes are enriched in the signaling molecules, receptors and ion channels responsible for input/output transformations in neurons, and the cell adhesion molecules that specify neuronal connectivity. Thus, changes in their expression could lead to changes in circuit level function. Specifically, elongation of long genes through mobile element insertions, occurring in the early embryo or in germ cells, likely creates a reservoir of genetic elements as fodder for regulatory innovation. Subsequent exaptation of a fraction of these elements could enhance cell type-, and hence, behavioral- diversity, in turn, increasing the ability of populations to adapt to their environment. If so, this helps explain the paradox of why long genes should be abundantly expressed in CNS neurons despite the fact that these genes are sites of genome instability associated with genetic lesions leading to autism and other developmental disorders \citep{Wei_2016}. 

This hypothesis also shifts focus away from short, developmental time scales considered in other hypotheses linking transposon insertion to neuronal function \citep{Muotri_2005,Richardson_2014,Perrat_2013}. Instead of DNA rearrangements in neuronal progenitors producing neuronal diversity, we consider the time scales of evolution and thus also shift focus to the germ line, where natural selection has its influence. 



