
\section{Discussion}

\subsection{A Resource of Neuronal Cell type specific Transcriptomes}
The dataset presented here is the largest collection of cell type-specific neuronal transcriptomes obtained by RNASeq (Table 1) and so offers the broadest view of the transcriptional basis of neuronal diversity. Unlike data sets obtained using microarray (Sugino 2005, Cahoy 2008, Heiman 2008, Doyle 2009, Siegert 2012), RNASeq permits discovery of unannotated genes and splice variants. Prior RNASeq data from sorted cells have been focused primarily on what distinguishes neurons as a class from other brain cell types (Zhang 2014), or have focused on a limited number of brain regions, such as the somatosensory cortex, hippocampus (Zeisel 2015) and retina (Macosko 2015). Our strategy of profiling labeled populations of ~100 cells is intermediate between single cell profiling, which can be limited by the noisiness of single cell assays (Marinov 2013) and tissue profiling, which cannot resolve the heterogeneity of component cell types. This approach enabled us to obtain highly sensitive and reproducible transcriptomes from genetically accessible target populations. The wide range of cell types in the dataset is suitable for addressing general questions regarding neuronal identity and diversity, but at the same time, the fact that each transcriptome corresponds to a genetically (or retrogradely) labeled population, which allows investigation of the same population of the cells across time and labs, should be useful for researchers with more specific questions about those cell types.

%% REMOVE
\subsection{New de novo RNASeq assembler}
When applied to the more than 500 libraries collected, two of the most popular and best performing existing RNASeq assembler programs (Hayer 2015), did not perform satisfactorily. The limitations arise primarily from the selection of junctions and quantification of transcripts (isoforms) and have been noted with other real and simulated datasets (Hayer 2015). Recognizing this, we first extracted exons and junctions independently from explicit transcript models, and then used additional information, such as branch probabilities to enumerate and quantify transcripts. The results improved the overlap with Gencode annotation relative to the existing programs tested.


%% REWRITE according to the RESULTS
\subsection{Diversity from Gene Expression}
We developed novel metrics to quantify transcriptome diversity. DI is a measure of the fraction of cell types distinguishable by a given gene or splice variant. As such, it is closely related to measures of information and highly correlated with mutual information between expression levels and cell types (Supplementary Figure 4C). Although no metric perfectly captures all features of the data, the metrics adopted allow us to assess the relative contribution of different classes of genes and types of transcripts to neuronal diversity. Analyses at the gene expression level indicated that ion channels, receptors and cell adhesion molecules are the groups of genes that contribute most to neuronal diversity. This is consistent with the fact that neuronal cell types' phenotypic diversity are closely tied to their electrical properties and synaptic connectivity. SC is a measure of the contrast, akin to the signal-to-noise ratio (SNR; actually SC=1-1/SNR). We found that the homeobox family of transcription factors are strikingly enriched among genes with high SC. Homebox and basic-helix-loop-helix (bHLH) transcription factors are known to be important in specifyn neuronal cell types (Bertrand 2002). For example, in the spinal cord, LIM homeobox genes are known to combinatorically regulate motor neuron identities (Tsuchida 1994, Shirasaki 2002) and in retina, homeobox gene Chx10 and bHLH genes Mash1 and Math3 are required for bipolar cell specification. In our dataset, the homeobox family of genes as a group has a combined DI of 0.98 (i.e. 98\% of the pairs can be distinguishable by homeobox gene expression), and together, homeobox and bHLH TFs have a combined DI of 0.99, indicating that all but ~1\% of pairs can be distinguishable by these two families of TFs. (Note that the dataset contains several Purkinje and Hippocampal pyramidal cell groups that may actually represent multiple examples of the same cell types.) The present data provides novel hypotheses of specific TF combinations from these two families in specifying neuronal cell types.

Genes that are universally expressed (EP=1) can also be highly informative with respect to cell type identity. For example, ribosomal proteins and other classic "housekeeping" genes vary widely in their expression level (Figure 4E) and this variation appears to correlate with features such as cell size. The observation that the cell types with high ribosomal gene expression are mostly large, highly secreting cells and those with low expression are small granule cells (Supplemental Figure 4D) indicates that the differences in their expression likely reflect adaptation to differences in cell types' needs to synthesize secreted and structural peptides and proteins.

\subsection{Splice differences}
Employing the new assembler, we were able to detect nearly two times the number of alternative junctions as are currently annotated (Supplemental Figure 2B). Traditionally alternative splicing has been analyzed against 4 to 5 limited number of patterns such as casette exon, mutually exclusive exons, alternative 3' end, alternative 5' end. Although being the most abundant classes, they are still minority of all the possible splicing patterns (Supplementary Figure 5G). By focusing on a simpler unit of "alternativeness" that is branches in splice graph, we were able to quantify splice difference in more general way. The newly detected branches tend to be expressed in very small number of samples (Supplementary Figure 5H left, also see NMD element in Figure 5E heatmap). Moreover, they are more enriched with NMD leading branches (Supplementary Figure 5H right). Therefore, some of these exons may correspond to "cryptic exons" described in (Eom 2013). Genes with junctions that are highly differentially spliced are enriched for G-protein modulators, non-receptor serine/threonine protein kinase, ion channels and mRNA processing factors (Figure 5G and Supplementary Figure 5C-F). G-protein modulator includes guanyl-nucleotide exchange factors such as RhoGEF or GTPase activating proteins such as RhoGAP and important in intracellular signaling pathways. Interestingly G-protein modulators turned out to be enriched in long genes that are expressed in both neuronal and non-neuronal cell types or just within any long genes (Supplemental Figure 6F,G).

\subsection{Neuronal genes are biased toward long genes which have higher capacity of differential expression}
One of the most robust findings in this study is that genes specific to neurons are biased toward long genes whereas genes not expressed in neurons lack this bias (Figure 6A). Long genes also contribute disproportionately to neuronal diversity, both at the level of gene expression and splicing. Increases in the number of alternative start and splice sites in longer genes may contribute, but in addition, we hypothesize that longer genes have a larger number of regulatory elements that alter expression and enhance differential usage of these alternative sites. Long genes likely elongate during evolution, via insertions of retrotransposons in their introns (Sela 2007, Grishkevich 2014). Long neuronal genes such as ion channels and cell adhesion molecules, may be expressed primarily late in development (Okaty 2009). Later and more restricted expression may make mammalian genomes more tolerant to mutations caused by the insertion of retrotransposons in these long genes. Conversely, genes such as Hox genes, that are critical for early development, and are often expressed in progenitors giving rise to many cell types, are remarkably transposon impoverished (Chinwalla 2002, Simons 2005). Transposon insertions occuring randomly are also expected to occur more frequently in long genes, thereby accelerating their elongation over the course of vertebrate evolution. This trend may contribute to the massive increase in repetitive sequences present in primate genomes relative to rodent genomes (ref).

Retroelements insertions have been reported to generate new promoters, exons and enhancers through a process called "exaptation" (for review see (Feschotte 2008)). Evolution of the vertebrate nervous system may have taken advantage of this process to diversify neuronal cell types increasing the complexity of brain circuits. Long genes are enriched in signaling molecules, receptors and ion channels responsible for input/output transformations neurons, and cell adhesion molecules that specify connectivity between neuronal cell types. Thus, changes in their expression could lead to changes in circuit level function. If so, this may help to explain the paradox of why long genes should be selectively expressed in CNS neurons despite the fact that these genes are sites prone to genomic instability leading to genetic lesions that cause autism and other developmental disorders (Wei 2016).

\subsection{Conclusion}
We present the most comprehensive dataset of neuronal and non-neuronal cell type-specific transcriptomes to date. Using this dataset, we show that neuronal genes are biased toward long genes. These long gene are more differentially expressed and thus contribute to the diversification of neuronal populations.
