
\section{Discussion}

\subsection{A Resource of Neuronal Cell type specific Transcriptomes}
The dataset presented here is the largest collection of cell type-specific neuronal transcriptomes obtained by RNASeq (Table 1) and so offers the broadest view of the transcriptional basis of neuronal diversity. Prior RNASeq data from sorted cells have been focused primarily on what distinguishes neurons as a class from other brain cell types \cite{Zhang_2014}, or have focused on a limited number of brain regions, such as the somatosensory cortex, hippocampus  \cite{Zeisel_2015} and retina \cite{Macosko_2015}. Our strategy of profiling labeled populations of ~100 cells is intermediate between single cell profiling, which can be limited by the noisiness of single cell assays \cite{Marinov_2013} and tissue profiling, which cannot resolve the heterogeneity of component cell types\cite{Nelson_2006}. This approach enabled us to obtain highly sensitive and reproducible transcriptomes from genetically accessible target populations. The wide range of cell types in the dataset is suitable for addressing general questions regarding neuronal identity and diversity, but at the same time, the fact that each transcriptome corresponds to a genetically (or retrogradely) labeled population, which allows investigation of the same population of the cells across time and labs, should be useful for researchers with more specific questions about those cell types.

We developed a quantitative approach for comparing cell type profiles across multiple studies using NNLS decomposition. The results reveal multiple cases in which pooled cell profiles mapped to more than one SCRS profile. It is likely that at least some of these cases represent biologically distinct cell types that share a genetic marker (like subtypes of Pvalb interneurons). However, in most of these cases, the SCRS clusters were barely separable, and the two SCRS studies available for comparison did not agree. Given the complimentary advantages of improved reproducibility and separability afforded the sorting approach, and of reduced heterogeneity afforded the SCRS approach, it is likely that further integration of these approaches with other modalities, such as FISH \cite{Moffitt_2016} will be needed to accurately profile the full census of brain cell types. 

%% REWRITE according to the RESULTS
\subsection{A transcriptional code for neuronal diversity}
We developed novel, easily calculated metrics that capture essential features of the robustness and information content of transcriptome diversity. These measures are not cleanly captured by traditional variance-based metrics like ANOVA and CV (Figure 3). We found that the homeobox family of transcription factors are strikingly enriched among genes with high SC. Homebox transcription factors are known to be important in specifying neuronal cell types. For example, in the vertebrate brainstem and spinal cord, homeobox genes are known to combinatorically regulate neuronal identity \cite{Dasen_2009}\cite{Philippidou_2013} as they do, in combination with other TFs in other bilaterians including drosophila and C. elegans \cite{Kratsios_2017}. In our data set, the homeobox family of genes as a group has a combined DI of 0.98, meaning that all but 2\% of the pairs can be distinguished by homeobox gene expression. (Note that the data set contains several Purkinje and Hippocampal pyramidal cell groups that may actually represent multiple examples of the same cell types).

\subsection{Long genes shape neuronal diversity}
One of the most robust findings in this study is that genes specific to neurons are biased toward long genes whereas genes not expressed in neurons lack this bias (Figure 6A). Long genes also contribute disproportionately to neuronal diversity, both at the level of gene expression and splicing. Increases in the number of alternative start and splice sites in longer genes may contribute, but in addition, we hypothesize that longer genes have a larger number of regulatory elements that alter expression and enhance differential usage of these alternative sites. Long genes likely elongate during evolution, via insertions of retrotransposons in their introns (Sela 2007, Grishkevich 2014). Long neuronal genes such as ion channels and cell adhesion molecules, may be expressed primarily late in development (Okaty 2009). Later and more restricted expression may make mammalian genomes more tolerant to mutations caused by the insertion of retrotransposons in these long genes. Conversely, genes such as Hox genes, that are critical for early development, and are often expressed in progenitors giving rise to many cell types, are remarkably transposon impoverished (Chinwalla 2002, Simons 2005). Transposon insertions occuring randomly are also expected to occur more frequently in long genes, thereby accelerating their elongation over the course of vertebrate evolution. This trend may contribute to the massive increase in repetitive sequences present in primate genomes relative to rodent genomes (ref).

Retroelement insertions have been reported to generate new promoters, exons and distal regulatory elements through a process called "exaptation" (for review see (Feschotte 2008)). For example, the set of neuronal genes regulated by REST has been suggested to have evolved through retrotransposon-mediated duplication\cite{Johnson_2006}.  Evolution of the vertebrate nervous system may have taken advantage of this process to diversify neuronal cell types increasing the complexity of brain circuits. Long genes are enriched in signaling molecules, receptors and ion channels responsible for input/output transformations neurons, and cell adhesion molecules that specify connectivity between neuronal cell types. Thus, changes in their expression could lead to changes in circuit level function. If so, this may help to explain the paradox of why long genes should be selectively expressed in CNS neurons despite the fact that these genes are sites prone to genomic instability leading to genetic lesions that cause autism and other developmental disorders (Wei 2016).

\subsection{Conclusion [REWRITE according to the RESULTS]}
We present the most comprehensive dataset of neuronal and non-neuronal cell type-specific transcriptomes to date. Using this dataset, we show that neuronal genes are biased toward long genes. These long gene are more differentially expressed and thus contribute to the diversification of neuronal populations.
