
\section{Discussion}

\subsection{A Resource of Neuronal Cell type specific Transcriptomes}
The dataset presented here is the largest collection of cell type-specific neuronal transcriptomes obtained by RNASeq (Table 1) and so offers the broadest view of the transcriptional basis of neuronal diversity. Prior RNASeq data from sorted cells have been focused primarily on what distinguishes neurons as a class from other brain cell types \cite{Zhang_2014}, or have focused on a limited number of brain regions, such as the somatosensory cortex, hippocampus  \cite{Zeisel_2015} and retina \cite{Macosko_2015}. Our strategy of profiling labeled populations of ~100 cells is intermediate between single cell profiling, which can be limited by the noisiness of single cell assays \cite{Marinov_2013} and tissue profiling, which cannot resolve the heterogeneity of component cell types \cite{Nelson_2006}. This approach enabled us to obtain highly sensitive and reproducible transcriptomes from genetically accessible target populations. The wide range of cell types in the dataset is suitable for addressing general questions regarding neuronal identity and diversity, but at the same time, the fact that each transcriptome corresponds to a genetically (or retrogradely) labeled population, which allows investigation of the same population of the cells across time and labs, should be useful for researchers with more specific questions about those cell types.

We developed a quantitative approach for comparing cell type profiles across multiple studies using NNLS decomposition. The results reveal multiple cases in which pooled cell profiles mapped to more than one SCRS profile. It is likely that at least some of these cases represent biologically distinct cell types that share a genetic marker (like subtypes of Pvalb interneurons). However, in most of these cases, the SCRS clusters were barely separable, and the two SCRS studies available for comparison did not agree. Given the complimentary advantages of improved reproducibility and separability afforded the sorting approach, and of reduced heterogeneity afforded the SCRS approach, it is likely that further integration of these approaches with other modalities, such as FISH \cite{Moffitt_2016} will be needed to accurately profile the full census of brain cell types. 

\subsection{A transcriptional code for neuronal diversity}
We developed novel, easily calculated metrics that capture essential features of the robustness and information content of transcriptome diversity. These measures are not cleanly captured by traditional variance-based metrics like ANOVA and CV (Figure 3). We found that the homeobox family of TFs exhibited the most robust (high SC) expression differences across cell types (Figure 4A). These “On/Off” differences were characterized by extremely low expression in the “Off” state (Figure 4C-D). Mechanistically, the low expression was associated with reduced genome accessibility measured by ATAC-seq (Figure 4E-F) presumably reflecting epigenetic regulation, known to occur for example at the clustered Hox genes via Polycomb group (PcG) proteins \cite{Montavon_2014}. Although this regulation has been studied most extensively at Hox genes, genome-wide ChIP studies reveal that PcG proteins are bound to over 100 homeobox TFs in ES cells \cite{Boyer_2006}. Our results indicate that strong cell type-specific repression persists in the adult brain. Presumably this represents the continued functional importance of preventing even partial activation of inappropriate programs of neuronal identity. 

As a group, homeobox TFs distinguished 98\% of neuronal cell types profiled. (Note this includes several Purkinje and Hippocampal pyramidal cell groups that may actually represent duplicate examples of the same cell types). Historically, homebox TFs are well known to combinatorically regulate neuronal identity in Drosophila and C. elegans \cite{Kratsios_2017} and the vertebrate brainstem and spinal cord \cite{Dasen_2009,Philippidou_2013}. Our results suggest a broader importance of homeobox TFs throughout the mammalian nervous system. Continued expression of these factors suggest they likely also contribute to the maintenance of neuronal identity.

In order to reveal the relationship between specific cell types and TFs, we constructed a TF decision tree for classifying profiled cell types. As expected from their high information content, homeobox TFs figured prominently in this list (49/127). Many of the identified factors are known to be key transcriptional regulators (KTRs) of the cell types in which they continue to be expressed (Supplemental table). In most cases it is not known whether or not these roles occur only in development, or are also important for the maintenance of neuronal identity. Lists of expressed TFs and the genetically accessible cell types in which they are expressed provide a ready source of testable hypotheses about how cell type specific transcriptional identity is maintained in the adult nervous system.

\subsection{Long genes shape neuronal diversity}
One of the most robust findings in this study is that long genes also contribute disproportionately to neuronal diversity (Figure 6A). Increases in the number of alternative start and splice sites in longer genes may contribute, but in addition, we hypothesize that longer genes have a larger number of regulatory elements that alter expression and enhance differential usage of these alternative sites. Long genes likely elongate during evolution, via insertions of retrotransposons in their introns \cite{Sela_2007,Grishkevich_2014}. Long neuronal genes such as ion channels and cell adhesion molecules, may be expressed primarily late in development \cite{Okaty_2009}. Later and more restricted expression may make mammalian genomes more tolerant to mutations caused by the insertion of retrotransposons in these long genes. Conversely, genes such as Hox genes, that are critical for early development, and are often expressed in progenitors giving rise to many cell types, are remarkably transposon impoverished \cite{Chinwalla_2002,Simons_2005}. Transposon insertions occuring randomly are also expected to occur more frequently in long genes, thereby accelerating their elongation over the course of vertebrate evolution. This trend may contribute to the massive increase in repetitive sequences present in primate genomes relative to rodent genomes \cite{Chuong_2016}. 

Retroelement insertions have been reported to generate new promoters, exons and distal regulatory elements through a process called "exaptation" (for review see \cite{Chuong_2016}). For example, the set of neuronal genes regulated by REST has been suggested to have evolved through retrotransposon-mediated duplication of REST binding sites \cite{Johnson_2006}. Similarly, TEs have helped distribute enhancers important for mammalian innate immunity \cite{Chuong_2016a}, shaped genome regulatory organization via distribution of the vertebrate insulator component CTCF\cite{Schmidt_2012} and helped diversify placental function across mammalian lineages \cite{Chuong_2013}. These innovations were originally seen as fortuitous benefits of an evolutionary arms race between the "selfish DNA" of TEs and the mechanisms by which host genomes suppress them. These mechanisms include the Kruppel-associated box (KRAB) domain-containing zinc-finger proteins, which are hugely expanded in mammals, especially in primates, where an evolutionary arms race has been explicitly documented \cite{Jacobs_2014}. Intriguingly, recent studies of these proteins indicate positive selection persisting long after the TEs they evolved to suppress lost their ability to transpose, suggesting that the arms race itself has been coopted for other regulatory purposes \cite{Imbeault_2017}. 

Here we provide evidence supporting the hypothesize that evolution of the vertebrate nervous system may have taken advantage of retrotransposition and subsequent exhaptation to diversify neuronal cell types, increasing the complexity of brain circuits. Long genes are enriched in the signaling molecules, receptors and ion channels responsible for input/output transformations in neurons, and the cell adhesion molecules that specify neuronal connectivity. Thus, changes in their expression could lead to changes in circuit level function. Specifically, elongation of long genes through TE insertions, occurring in the early embryo or in germ cells, created a reservoir of genetic elements providing fodder for regulatory innovation. Subsequent exaptation of a fraction of these elements may have enhanced cell type-, and hence, behavioral- diversity, in turn, increasing the ability of populations to adapt to their environment. If so, this may help to explain the paradox of why long genes should be abundantly expressed in CNS neurons despite the fact that these genes are sites of genome instability associated with genetic lesions leading to autism and other developmental disorders \cite{Wei_2016}. 

This hypothesis shifts focus away from short, developmental time scales considered in other hypotheses linking gene length or transposon insertion to neuronal function \cite{Muotri_2005}\cite{Richardson_2014}\cite{Perrat_2013}. Instead of DNA rearrangements in neuronal progenitors producing neuronal diversity, we consider the time scales of evolution and thus also shift focus to the germ line, where natural selection has its influence. 

