\section*{Introduction}

% - remove assembler part
% - add homeobox
% - add regional difference etc.

The extraordinary diversity of vertebrate neurons has been appreciated since the proposal of the neuron doctrine (Cajal, 1995). Typically, this diversity is characterized by neuronal morphology, physiology, molecular expression, and circuit connectivity. The exact number of neuronal cell types remains unknown, but estimates of 40-60 have been provided for the retina (Macosko et al., 2015; Masland, 2004) and for mouse cortex (Tasic et al., 2016; Zeisel et al., 2015). If similar numbers are discovered in most brain regions, the number could be in the thousands or more. Although neuronal diversity has long been recognized, the question of how this diversity arises has just started to be asked (Arendt, 2008; Muotri and Gage, 2006). Describing the cell types of the brain and understanding the principles governing their diversity are important goals of neuroscience.

Existing studies have used SCRS (Tasic, Zeisel,hypothlamus papers), but have been limited to single regions and and suffer from noise issues. No systematic effort has been undertaken to relate these studies to one another.
Recently, single-cell RNA-seq (SCRS) has emerged as the preferred method for profiling transcriptional diversity in neurons () and other cell types (). An obvious advantage of the SCRS approach is that, by definition, each measurement comes from only a single cell type. However, SCRS measurements can be noisy, and, depending on the approach, they may have limited depth and sensitivity (). Noise can arise technically from amplification, or biologically from the stochastic nature of transcription. It is hoped that accurate and precise transcriptional profiles of cell types can be obtained by clustering the profiles of single cells and then averaging, but the process of clustering itself can add noise (), and the unbiased nature of the measurement complicates the assessment of reproducibility. To provide a complementary view of cell type-specific transcription, we profiled genetically identified neurons (GENSAT, Shima et al. 2016) by deep RNA sequencing (RNA-seq) of manually sorted fluorescent neurons from micro-dissected brain regions.  

Using this approach, we determined the complete gene expression profiles for 174 neuronal cell types and 15 non-neuronal cell types and quantitatively compared the profiles of cortical cell types to those obtained in SCRS studies. The comparison reveals a comparable level of homogeneity, but a much lower level of noise in the sorted profiles. These reproducible and precise expression profiles serve as a look-up table for linking single cell and cell type expression profiles to genetic strains in which they can be repeatedly accessed. 

The ability to detect differences in expression between cell types depends not only on the noise of the measurement, but on the signal-to-noise (SNR) of the underlying gene expression. Using a simple measure related to the SNR, we find that homeobox transcription factors (TFs) have the highest SNR of any gene family. We also introduce an easily calculated metric, termed the Differentiation Index (DI), that is closely related to the mutual information between genes and cell types. In addition to having low-noise expression, homeobox TFs have the highest mutual information with respect to cell types. Using homeobox and other highly informative TFs we construct a compact “code” for profiled neuronal cell types and find that the identified genes are highly enriched in genes known to be key transcriptional regulators of neuronal cell types.
Computing DI over individual genes, we find that the genes carrying the most information about cell types are neuronal effector genes like receptors, ion channels and cell adhesion molecules. Two common feature of these genes are their genomic length, reflecting the number and length of their introns, and their regulation outside of the nervous system by the repressor REST.

Long genes have recently been recognized as sites of genomic instability (Wei et al., 2016), and as contributors to multiple nervous system diseases (Sugino et al., 2014; Gabel et al., 2015; Zylka et al., 2015). Given these hazards and the fact that long genes are metabolically expensive to produce (Castillo-Davis et al., 2002), why should their expression be so prominent in the brain? Our results suggest their length may permit increased complexity in their regulation, thereby enhancing the diversity of neuronal cell types. Consistent with this idea, we find that long genes have larger numbers of candidate regulatory regions identified as sites of genome accessibility in ATAC-seq experiments, and that these regions are present in a larger number of patterns across neuronal cell types than candidate regulatory regions in short genes.
 