\section*{Introduction}

The extraordinary diversity of vertebrate neurons has been appreciated since the proposal of the neuron doctrine \cite{0195074017}. Typically, this diversity is characterized by neuronal morphology, physiology, molecular expression, and circuit connectivity. The exact number of neuronal cell types remains unknown, but estimates of 40-60 have been provided for the retina \cite{Macosko_2015,Masland_2004} and for mouse cortex \cite{Tasic_2016,Zeisel_2015}. If similar numbers are discovered in most brain regions, the number could be in the thousands or more. Although neuronal diversity has long been recognized, the question of how this diversity arises is only beginning to be asked \cite{Arendt_2008,Muotri_2006}. Describing the cell types of the brain and understanding the principles governing their diversity are fundamental goals for neuroscience.

Recently, single-cell RNA-seq (SCRS) has emerged as the preferred method for profiling transcriptional diversity in neurons and other cell types \cite{Shapiro_2013}. An obvious advantage of the SCRS approach is that, by definition, each measurement comes from only a single cell type. However, SCRS measurements can be noisy, and, depending on the approach, they can have limited depth and sensitivity \cite{Parekh_2016,Svensson_2017}. Noise can arise technically from amplification, or biologically from the stochastic nature of transcription. So far, the field attmepts to generate accurate and precise transcriptional profiles of cell types by clustering and then averaging the profiles of single cells. But the process of clustering itself can add noise \cite{Ntranos_2016}, and the unbiased nature of the measurement complicates the assessment of reproducibility. To provide a complementary view of cell type-specific transcription, we used deep RNA sequencing (RNA-seq) to profile the largest set to date of genetically identified and fluorescently labeled pooled neurons from micro-dissected brain regions.

Using this approach, we determined the complete gene expression profiles for 178 neuronal cell types and 15 non-neuronal cell types and quantitatively compared our cortical profiles to those obtained in SCRS studies. The comparison reveals a comparable level of homogeneity, but a much lower level of noise in the bulk sorted profiles. We have curated these reproducible and precise expression profiles to serve as a look-up table for linking single cell and cell type expression profiles to genetic strains in which they can be repeatedly accessed. 

%previous version:
%Standard methods for differential expression of RNAseq data focus mainly on signal variance but are influenced by both information content and robustness of differential expression. To separate these two, we defined two simple measures that separately quantify these two measures of differential gene expression. Using , we find that homeobox transcription factors (TFs) have the lowest off-noise of any gene family. We also introduce an easily calculated metric, termed the Differentiation Index (DI), that is closely related to the mutual information between genes and cell types. In addition to having low-noise expression, homeobox TF family has the highest mutual information with respect to cell types. Using DI to identify other highly informative TFs, we construct a compact “code”, similar to a decision tree, for profiled neuronal cell types. The vast majority of identified TFs turn out to be known key transcriptional regulators of neuronal identity.

Standard methods for differential expression of RNAseq data focus mainly on signal variance but are influenced by both information content and robustness of differential expression. We introduced two simple metrics to address this problem. Signal contrast (SC) is a signal-to-noise ratio that (unlike ANOVA) is not sensitive to differences in information content. Differentiation index (DI) is a measure of information content closely related to mutual information. Using SC, we find that homeobox transcription factors have the lowest off-noise of any gene family as a result of closed chromatin in the promoter and gene body of these genes. Using a combination DI and SC to identify other highly informative TFs, we construct a compact “code”, similar to a decision tree, for profiled neuronal cell types. The vast majority of identified TFs turn out to be known key transcriptional regulators of neuronal identity.

We find that the genes carrying the most information about cell types are neuronal effector genes like receptors, ion channels and cell adhesion molecules. We find two common feature of these genes are their genomic length, reflecting the number and length of their introns, and their regulation outside of the nervous system by the repressor REST. Long genes such as these have recently been recognized as sites of genomic instability \cite{Wei_2016}, and as contributors to multiple nervous system diseases \cite{Sugino_2014,Gabel_2015,Zylka_2015}. Given these hazards and the fact that long genes are metabolically expensive to produce \cite{Castillo_Davis_2002}, why should they be so prominent in the brain? Our results suggest their length may permit increased regulatory complexity, with long genes containing a larger number of candidate regulatory regions identified by ATAC-seq arrayed in more diverse patterns across neuronal cell types than found in short genes. Genes with neuronal function may be more likely to grow long as the noncoding sequences provide a platform for evolution to fine-tune gene expression. This fine-tuning could lead to cellular diversification, which we know is a critical aspect of brain and behavioral complexity.
