\section*{Introduction}

The extraordinary diversity of vertebrate neurons has been appreciated since the proposal of the neuron doctrine \citep{Cajal_1888}. Typically, this diversity is characterized by neuronal morphology, physiology, molecular expression, and circuit connectivity. The exact number of neuronal cell types remains unknown, but estimates of 40-60 have been provided for the retina \citep{Macosko_2015,Masland_2004} and for mouse cortex \citep{Tasic_2016,Zeisel_2015}. If similar numbers are discovered in most brain regions, the number could be in the thousands or more. Although neuronal diversity has long been recognized, the question of how this diversity arises is only beginning to be addressed \citep{Arendt_2008,Muotri_2006}. Describing the cell types of the brain and understanding the principles governing their diversity are fundamental goals for neuroscience.

Currently two techniques dominate the efforts to profile the transcriptional diversity of cell types in the brain: one is RNA-seq from single neurons, (single-cell RNA-seq; SCRS), \citep[e.g.][]{Shapiro_2013} and the other is from genetically or anatomically marked pools of neurons \citep[e.g.][]{Okaty_2015,Cembrowski_2016}.  An obvious advantage of the SCRS approach is that, by definition, each measurement comes from only a single cell type. However, SCRS measurements can be noisy and, depending on the approach, can have limited depth and sensitivity \citep{Parekh_2016,Svensson_2017}. So far, the field attempts to generate accurate and precise transcriptional profiles of cell types by clustering and then averaging the profiles of single cells. But the process of clustering itself can add noise \citep{Ntranos_2016}, and the unbiased nature of the measurement complicates the assessment of reproducibility. Pooling reduces noise, but can suffer from unknowingly lumping together more than one cell type. In the end, performing both methods will allow for a more confident assessment of the cell types of the brain. While large, unbiased single cell efforts have been completed or are underway, similar large scale efforts for genetically identified neurons have yet to be reported. We performed RNA-seq on the largest set to date of genetically identified and fluorescently labeled pooled neurons from micro-dissected brain regions. In total, we profiled 179 neuronal cell types and 15 non-neuronal cell types and quantitatively compared our cortical profiles to those obtained in SCRS studies. (A more precise description of our use of the term "cell type" is provided in the Methods). The comparison reveals a comparable level of homogeneity, but a much lower level of noise in the bulk sorted profiles. We have curated these reproducible and precise expression profiles to serve as a look-up table for linking single cell and cell type expression profiles to genetic strains in which they can be repeatedly accessed. 

Cell types are typically identified by performing differential expression analyses. Standard differential expression methods focus on signal variance but are influenced by both information content and robustness of differential expression. We introduced two simple metrics to separate out these features of the data. Signal contrast (SC) is a signal-to-noise ratio that (unlike ANOVA) is not sensitive to differences in information content. Differentiation index (DI) is a measure of information content closely related to mutual information. Using these metrics, we identify homeobox transcription factors (TF) as the gene family with the lowest noise and highest ability to distinguish cell types and use these and other TFs to construct a compact “code” for profiled neuronal cell types. We find that the effector genes carrying the most information about cell types are synaptic genes like receptors, ion channels and cell adhesion molecules. Interestingly, a common feature of these genes is their long genomic length, reflecting the increased number and length of their introns. 
%Long genes such as these have recently been recognized as sites of genomic instability \cite{Wei_2016}, and as contributors to multiple nervous system diseases \cite{Sugino_2014,Gabel_2015,Zylka_2015}. Given these hazards and the fact that long genes are metabolically expensive to produce \cite{Castillo_Davis_2002}, why should they be so prominent in the brain? 
Our ATAC-seq results indicate that long genes contain a larger number of candidate regulatory regions which are arrayed in more diverse patterns than found in short genes, suggesting the longer length of the genes may permit increased regulatory complexity.
%Our results suggest their length may permit increased regulatory complexity, with long genes containing a larger number of candidate regulatory regions identified by ATAC-seq arrayed in more diverse patterns across neuronal cell types than found in short genes. 
Moreover, these long genes are elongated during evolution by insertions of mobile elements and a large portion of the candidate regulatory regions identified by ATAC-seq overlap with these mobile elements. Thus, the increased length of neuronal genes may provide a platform for evolution to fine-tune gene expression and thus diversify the cell types of the nervous system.
