\section{Methods and Materials}

\subsection{Mouse Lines}
Mouse lines profiled in this study are summarized in Supplementary Table 1. Most were obtained from GENSAT \citep{Gong_2007} or from the Brandeis Enhancer Trap Collection \citep{Shima_2016}. For Cre-driver lines, the Ai3, Ai9 or Ai14 reporter \citep{Madisen_2009} was crossed and offspring hemizygous for Cre and the reporter gene were used for profiling. All experiments were conducted in accordance with the requirements of the Institutional Animal Care and Use Committees at Janelia Research Campus and Brandeis University.

\subsection{Atlas}
Animals were anesthetized and perfused with 4\% paraformaldehyde and brains were sectioned with 50$\mu m$ thickness. Every fourth section was mounted on slides and imaged with a slide scanner equipped with 20x objective lens (3DHISTECH; Budapest, Hungary). In house programs were used to adjust contrast and remove shading caused by uneven lighting. Images were converted to a zoomify compatible format for web delivery and are available at http://neuroseq.janelia.org.

\subsection{Cell Sorting}
Manual cell sorting was performed as described before \citep{Hempel_2007, Sugino_2014}. Briefly, animals were sacrificed following isoflurane anesthesia, and 300$\mu m$ slices were digested with pronase E (1mg/ml, P5147; Sigma-Aldrich) for 1 hour at room temperature, in artificial cerebrospinal fluid (ACSF) containing 6,7-dinitroquinoxaline-2,3-dione (20$\mu M$; Sigma-Aldrich), D-(–)-2-amino-5-phosphonovaleric acid (50$\mu M$; Sigma-Aldrich), and tetrodotoxin (0.1$\mu M$; Alomone Labs). Desired brain regions were micro-dissected and triturated with Pasteur pipettes of decreasing tip size. Dissociated cell suspensions were diluted 5-20 fold with filtered ACSF containing fetal bovine serum (1\%; HyClone) and poured over Petri dishes coated with Sylgard (Dow Corning). For dim cells, Petri dishes with glass bottoms were used. Fluorescent cells were aspirated into a micropipette (tip diameter 30-50$\mu m$ under a fluorescent stereomicroscope (M165FC; Leica), and were washed 3 times by transferring to clean dishes. After the final wash, pure samples were aspirated in a small volume (1$\sim$3$\mu l$) and lysed in 47$\mu l$ XB lysis buffer (Picopure Kit, KIT0204; ThermoFisher) in 200$\mu l$ PCR tube (Axygen), incubated for 30min at 40$^{\circ}$ on a thermal cycler and then stored at -80$^{\circ}C$. Detailed information on profiled cell groups are provided in Supplementary Table 2.

\subsection{RNA-seq}
Total RNA was extracted using Picopure kit (KIT0204; ThermoFisher). Either 1$\mu l$ of $10^{-5}$ dilution of ERCC spike-in control (\#4456740; Life Technologies) or (number of sorted cells/50) * (1$\mu l$ of $10^{-5}$ dilution of ERCC) was added to the purified RNA and speed-vacuum concentrated down to 5$\mu l$ and immediately processed for reverse transcription using the NuGEN Ovation RNA-Seq System V2 (\#7102; NuGEN) which yielded 4$\sim$8$\mu g$ of amplified DNA. Amplified DNA was fragmented (Covaris E220) to an average of $\sim$200bp and ligated to Illumina sequencing adaptors with the Encore Rapid Kit (0314; NuGEN). Libraries were quantified with the KAPA Library Quant Kit (KAPA Biosystems) and sequenced on an Illumina HiSeq 2500 with 4 to 32-fold multiplexing (single end, usually 100bp read length, see Supplemental Table 2).

\subsection{RNA-seq analysis}
Adaptor sequences (AGATCGGAAGAGCACACGTCTGAACTCCAGTCAC for Illumina sequencing and CTTTGTGTTTGA for NuGEN SPIA) were removed from de-multiplexed FASTQ data using cutadapt v1.7.1 (http://dx.doi.org/10.14806/ej.17.1.200) with parameters “--overlap=7 --minimum-length=30”. Abundant sequences (ribosomal RNA, mitochondrial, Illumina phiX and low complexity sequences) were detected using bowtie2 \citep{Langmead_2012} v2.1.0 with default parameters. The remaining reads were mapped to UCSC mm10 genome using STAR \citep{Dobin_2012} v2.4.0i with parameters “--chimSegmentMin 15 --outFilterMismatchNmax 3”. Mapped reads are quantified with HTSeq \citep{Anders_2014} using Gencode.vM13 \citep{Harrow_2012}.

\subsection{Pan-neuronal genes}
Pan-neuronal genes are extracted as satisfying following conditions: 1) mean neuronal expression level (NE)$>$ 20 FPKM, 2) minimum NE $>$ 5 FPKM, 3) mean NE $>$ maximum non-neuronal expression level (NNE), 4) minimum NE $>$ mean NNE, 5) mean NE $>$ 4x mean NNE, 6) mean NE $>$ mean NNE $+$ 2x standard deviation of NNE, 7) mean NE $-$ 2x standard deviation of NE $>$ mean NNE. 

\subsection{DI/SC/DN calculation}
To calculate q-values, the limma package with voom method \citep{Law_2014} is used. To adjust the power, two replicates (the newest two) are used for all cell types with more than two replicates. We have tried same calculations with 3 replicates (with fewer number of cell types) and obtained similar results (data not shown). 

For DM criteria for DI, log fold change $>$ 2 and q-value $<$0.05 is used. For binary DI (bDI), DM criteria is: expression levels of all the replicates in one of the cell types in the pair $<$ 1FPKM and expression levels of all the replicates in the other cell type in the pair $>$ 15FPKM. 

To assess the extent of differentiation by alternative splicing, we calculate differentiation at splice branch level. See Figure 3 Supplement 1B for the definition of branch, and branch probability. For each branch of alternative splice sites, and a pair of cell types, we call they are "different" when 1) either all the replicates from one group are less than 0.3 and all the replicates from the other group are greater than 0.7, or vice versa and 2) both cell types in the pair have reasonable reads at the alternative site ($>10$ reads). Condition 1) is justified by the bimodal distribution shown in Figure 3 Supplement 1C. Accumulating all pairs, these create a DM for each branch. We then combine all the branches to create a new DM for each gene (if any branch distinguishes a pair that pair is called "different" at the gene level). This DM represents how pairs of cell types are distinguished by any branches belonging to the gene. Since number of pairs actually compared may be different depending on gene (since some genes in some cell types may not be expressed), instead of DI which assumes fixed number of total pairs, we use DN (total number of pairs distinguished) to rank genes.

\subsection{NNLS decomposition}
SCRS datasets deposited in NCBI GEO (GSE71585, \cite{Tasic_2016}; GSE60361, \cite{Zeisel_2015}) were used for NNLS decomposition. Specifically the deposited count data are converted to TPM and used for comparison. The NeuroSeq dataset is quantified using RefSeq and featurecount \citep{Liao_2013} and converted into TPM. Subset of genes common to all three datasets are then used. Since distribution of TPM values were different between datasets, they are quantile normalized to an average profile generated from the NeuroSeq dataset. Since most of the genes in the SCRS profiles exhibited noisy pattern, using entire gene set for decomposition is not feasible. Therefore, we selected genes informative for decomposition (informative for distinguishing cell classes) by ANOVA using cell classes as groups. However, simply taking top ANOVA genes lead to biased selection of genes (e.g. many genes were specific to microglia), so starting from the highest ANOVA gene (highest ANOVA F-value), following genes are not selected if their DM (Differentiation Matrix defined in Figure 3) are close to that of any of the previously selected genes where closeness is measured by Jaccard index (threshold 0.5). We chose 300 from each dataset which yielded total of 563 genes when combined. This gene set is then used for all the decompositions. For SCRS, author's cluster assignments were used as the classification of single cell profiles into cell classes and to create average profiles which form the basis of the decomposition. NNLS implementation in Python scipy library \citep{Jones_2001} is used to solve the non-negative linear system. For validation of NNLS decomposition, see Figure 2 Supplements 1-3. 

\subsection{ATAC-seq}
7 cell types (Purkinje and granule cells from cerebellum, excitatory layer 5 and 6 cells from cortex, exitatory entorhinal, CA1, CA1-3 cells from hippocampus, labeled in mouse lines XX,XX,XX,...and XX respectively) were FACS sorted to obtain ~XXXX labeled neurons. ATAC libraries for Illumina next-generation sequencing were prepared in accordance with a published protocol \citep{Buenrostro_2013}. Briefly, collected cells were then lysed in XX and nuclei isolated [HOW] for Tn5 digestion. Library quality was assessed prior to sequencing using BioAnalyzer estimates of fragment size distributions (~XXXbp) as well as qPCR to determine relative enrichment at several common ATAC peaks (specifically XXX are assessed). For sequencing, Illumina HiSeq 2500 with XX to XX fold multiplexing with single read 100bp read length was used. In addition to ATAC-seq, RNA-seq were also performed on replicate samples collected in a similar way using the same library preparation method described above. 

\subsection{ATAC-seq analysis}
Nextera adaptors (CTGTCTCTTATACACATCT) were trimmed from both ends from de-multiplexed FASTQ files using cutadapt with parameters "-n 3 -q 30,30 -m 36". Reads were then mapped to UCSC mm10 genome using bowtie2 \citep{Langmead_2012} with parameters "-X2000 --no-mixed --no-discordant". PCR duplicates were removed using Picard tools (http://broadinstitute.github.io/picard, v2.8.1) and reads mapping to mitochondrial DNA, scaffolds, and alternate loci were discarded. BigWig genomic coverage files were generated using bedtools \citep{Quinlan_2010} and scaled by the total number of reads per million. For reproducible peaks, liberal peaks were called using HOMER (v4.8.3) \citep{Heinz_2010} with parameters "-style factor -region -size 90 -fragLength 90 -minDist 50 -tbp 0 -L 2 -localSize 5000 -fdr 0.5" and filtered using the Irreproducibility Discovery Rate (IDR) in homer-idr (http://github.com/karmel/homer-idr.git) with parameters "--threshold 0.05 --pooled-threshold 0.0125". Peak counts and peak patterns were then quantified using bedtools.

