\subsection{Gene length correlates with nervous system complexity}

The above results indicate that gene length is an important contributor to gene expression diversity across cell types. Cell type diversity is a hallmark of the nervous system, and is thought to be critical for its ability to generate complex behavioral outputs. Thus gene length and the mechanisms that drive lengthening over time are potential contributors to the evolution of nervous system complexity. This predicts that species with longer genes will have a larger, more diverse set of neuronal cell types, permitting construction of more complex neural circuits. Consistent with this, we find that the distribution of gene lengths for the best annotated species is broadly correlated with nervous system size and complexity (Figure 7A). This is also true of neuronal genes, (defined in the mouse), which also increase in length with increased nervous system complexity (Figure 7A). An example gene illustrating this relationship (the voltage-gated potasium channel Kcnma1 (slo)) is shown in Figure 7B. 

Vertebrate introns are known to have undergone significant expansion through multiple rounds of transposition, and this process is known to have accelerated during mammalian evolution, and more specifically, during primate evolution\cite{Friedli_2015}. Transposons are often species specific, and waves of transposon intertions and subsequent exhaptation are known to have contributed to species specific adaptations\cite{Villar_2015}\cite{Mita_2016}\cite{Imbeault_2017}, especially in primates\cite{Ward_2013}\cite{Han_2007}, where the events have occurred more recently and so can be traced more easily. 

To map species-specific genomic insertions potentially contributing to the elongation of human and mouse genes, we analyzed the genomic sequence differences between the closely related primate genomes of human and chimp and the closely related rodent genomes of mouse and rat. In each case, we used a third species (gorilla for human and guinea pig for mouse) to identify ancestral sequences, allowing more recent insertions in each lineage to be identified. When the histogram of the lengths of the inserted intervals are plotted, two peaks are apparent (Fig.7C). The shorter peak corresponds to Alu insertions and longer peak corresponds to L1 insertions. Overall, 92\% of the inserted base pairs in human and 85\% of those in mouse overlap annotated mobile elements \cite{Hubley_2015}. This and analyses from other groups \cite{Grishkevich_2014} indicate that genes are elongated by insertion of the mobile elements during evolution. Since different species have different sets of mobile elements, it is probable that specific set of mobile elements contributed more in elongation of genes.

