\subsection{Long genes contribute disproportionately to neuronal diversity}

Gene expression is known to be controlled at the chromatin level, by a combination of TF at the promoter region, or by enhancer/repressor elements. We reasoned that increased ability of high DI genes to be differentially expressed may be dependent on the relevant number of these cis-regulatory elements. Since these elements need to be isolated to each other in the genome, we inferred that longer genes may have higher capacity to be differentially expressed. This is in fact the case and longer genes have on average higher DIs (Fig.6A).

Recently, long genes have been found to be enriched in neurons and are frequently linked to developmental disorders of the CNS (Gabel et al., 2015; Sugino et al., 2014; Wei et al., 2016; Zylka et al., 2015). Consistent with these, in our dataset we observed that neurons express more long genes compared to non-neurons (Fig.6B). Since neuronal genes are often repressed by REST/NRSF in non-neurons, we wondered whether REST targets long genes preferentially. When length of the genes nearest to computationally detected REST elements (ref) are plotted, we observed that they are indeed biased toward long genes (Fig.6C). When these REST targets are removed from neuronally expressed genes, the length distribution looks similar to that of non-neurons (Fig.6D).

To find candidate regulatory elements responsible for cell-type specific gene expression, we performed high-throughput sequencing of open chromatin using ATAC-seq in 7 cell types and quantified the number of open chromatin sites as a surrogate for regulatory elements. As expected, long genes had more open chromatin sites (suppl) and a higher number of peak patterns per gene across cell types (defined as the number of uniquely present peak combinations per gene;  Fig. 6E), and these number of unique patterns correlated with the amount of differential expression as assesssed by DI and ANOVA (fig. 6F include ANOVA).
To test if the use of regulatory elements in long genes was universal or restricted to neurons, we used publicly available DNase-seq data from the ENCODE project (ref), particularly pronounced in forebrain. We found a significantly higher number of open chromatin sites in brain compared to non-brain tissue (sup.fig., sup.table). This bias was even stronger in human tissue compared to mouse tissue.

To further assess particular cell types were more biased towards long genes than others, we compared median gene expression across our cell types. The long gene bias was particularly strong in telencephalic regions, intermediate in midbrain, and less in hindbrain and cerebellum.... () [refer to mouse brain-projected heatmap, to be genrated, hypothesize about mechanism, e.g. Foxg1 or REST co-factors].