\subsection{Long genes contribute disproportionately to neuronal diversity}

Neuronal effector genes such as ion channels, receptors and cell adhesion molecules are known to be long genes that are enriched in neurons and frequently linked to CNS developmental disorders (Gabel et al., 2015; Sugino et al., 2014; Wei et al., 2016; Zylka et al., 2015). We found that these gene categories have the greatest ability to distinguish cell types (highest DI; Figure 4A). Consistent with this, DI is strongly length biased (Fig.6A) and neurons express more long genes compared to non-neurons (Fig.6B). Hence long genes are both preferentially expressed in neurons and contribute most to the differential expression between neuronal cell types. 

One likely reason for the differential expression of long genes in neurons is the well known repression of neuronally expressed genes by REST/NRSF in non-neurons (). To assess the magnitude of this effect and its influence on the length distribution of neuronal genes, we plotted the length-dependence of genes containing NRSE elements and observed that they are indeed biased toward long genes (Fig.6C). When these REST targets are removed from neuronally expressed genes, the length distribution looks similar to that of non-neurons (Fig.6D) and the length bias of DI flattened(???).

Long

To find candidate regulatory elements responsible for cell-type specific gene expression, we performed high-throughput sequencing of open chromatin using ATAC-seq in 7 cell types and quantified the number of ATAC-seq peaks as a surrogate for regulatory elements. As expected, long genes had more open chromatin sites (suppl) and a higher number of peak patterns per gene across cell types (defined as the number of uniquely present peak combinations per gene;  Fig. 6E), and the number of unique patterns correlated with the amount of differential expression as assessed by DI and ANOVA (fig. 6F include ANOVA).
To compare usage of regulatory elements in long genes between neurons and non-neurons, we used publicly available DNase-seq data from the ENCODE project (ref). We found a significantly higher number of open chromatin sites in brain compared to non-brain tissue (sup.fig., sup.table). This bias was  particularly pronounced in forebrain, and was even stronger in human tissue compared to mouse tissue (sub 6C).

To further assess if particular cell types were more biased towards long genes than others, we compared median gene expression across our cell types. The long gene bias was especially strong in telencephalic regions, intermediate in midbrain, and less in hindbrain and cerebellum.... () [refer to mouse brain-projected heatmap, to be genrated, hypothesize about mechanism, e.g. Foxg1 or REST co-factors].