\subsection{Information-based metrics to quantify differential expression}
In order to extract the transcriptional signals related to cell type identity, we quantified each gene's ability to differentiate each pair of profiled cell types. Based on expression levels and variability (Figure 3A; Methods) we compiled a Differentiation Matrix (DM) with elements equal to one or zero depending on whether or not the gene is differentially expressed between each pair of profiles. The Differentiation Index, DI, is simply the fraction of pairs distinguished, excluding self-comparisons; and ranges from 0 to 1. The maximum observed value of 0.65 indicates that the gene distinguishes 65\% of the pairs, while a value of 0 indicates that the gene distinguishes none (i.e., expressed at similar levels in all cells). 

The ability to detect transcriptional differences between cell types depends not only on the magnitudes of these differences, but also on their associated noise. To quantify this, we defined the Signal Contrast (SC), a metric closely related to the Signal-to-Noise-Ratio (SNR). Since the signals we are interested in are the gene expression differences distinguishing cell types, we used a noise estimate derived from all undistinguished pairs. SC, which indicates how robustly pairs are distinguished, is the ratio of the average effect size for distinguished and undistinguished pairs. 

To illustrate the properties of DI,SC relative to existing metrics such as ANOVA, coefficient of variation (CV) and mutual information (MI), we calculated these metrics against various simulated expression patterns with added noise (Figure 3B). The results (Figure 3B, lower) demonstrate that DI is highly correlated with MI. This makes intuitive sense, since the division of cell types into those that can and cannot be distinguished (DM; Figure 3A) corresponds to a unit of information about cell types provided by a gene expression pattern. As shown below (Methods), in the simplified case of binary expression levels and ignoring noise, DI is an order-preserved transformation of MI. More generally, DI is highly correlated with MI and can be regarded as an approximation to MI (For a related metric see Simpson's index of diversity [REF:1949 Nature, Measurement of Diversity]. Thus, DI provides an estimate of the mutual information between expression patterns and cell types contained, that can be rapidly and easily calculated. 

The simulations also show that DI is not well correlated with SNR. For example, high SNR graded patterns correspond to high DI (Figure 3B, lower right), while high SNR specific binary patterns correspond to low DI (Figure 3B, upper left). In contrast, SC is highly correlated to SNR, to which it is closely related. 

Unlike  is the traditional statistics of choice when quantitating differential expression of multiple cell types. Fig.2B indicates that ANOVA F-values are affected both by information content and signal-to-noise ratio. Therefore, when these aspects need to be separated, ANOVA is not an appropriate metric. For example, ANOVA F-value for Binary high SNR/Low MI is similar to Binary low SNR/High MI (Fig.2B), and they are not distinguishable. 

CV or VMR is often used to extract highly varying genes when no cell type labeling is available, such as in the case of single cell experiments to classify samples. From Fig.2B, we can see that CV is insensitive to information content nor SNR, VMR is sensitive to SNR but not information content.  Since it is desirable to have high information content as well as high signal-to-noise ratio for classification features, high DI/high SC genes are more appropriate for this purpose than high CV, or high VMR genes. 

Finally, Fig.2B indicates that SC captures SNR faithfully for binary cases and approximately for graded cases. This is enabled by separation of pairs in two groups using DM (Fig.2A), which is also used to calculate DI. High SC genes robustly distinguish cell populations, that is, they are suitable for "marker genes".  These feature differences between metrics are summarized in Fig.2C.

In Fig.D, relationships between DI/SC/ANOVA p-values calculated from the current dataset are shown. This also illustrates that highly  ANOVA significant genes consist of both high DI and high SC genes. 

In summary, DI and SC enable to independently measure the amount of differential expression, which can also be regarded as information content of cell types in the expression pattern, and the robustness of differential expression.