\subsection{Gene length correlates with nervous system complexity}
Vertebrate introns are known to have undergone significant expansion through multiple rounds of transposition \cite{Grishkevich_2014}, and this process has accelerated during mammalian evolution, and more specifically, during primate evolution \cite{Friedli_2015}. Recent estimates suggest that half to two thirds of the human genome may be derived from mobile elements \cite{Friedli_2015}. %[#wrong ref?No, correct]. 
To investigate the relationships between elongation via transposition and neuronal gene regulation, we first mapped genomic insertions potentially contributing to the elongation of genes using the genomic sequence differences between the closely related primate genomes of human and chimp and the closely related rodent genomes of mouse and rat. In each case, we used a third species (gorilla for human and chinese hamster for mouse) to identify ancestral sequences, allowing more recent insertions in each lineage to be identified. When the histograms of the lengths of the inserted intervals are plotted, two peaks are apparent (Figure 5A). 
%The shorter peak corresponds to Alu insertions and the longer peak corresponds to L1 insertions. [Add this to legend]
Overall, 88\% of the inserted base pairs (between 100bp and 100kbp in size) in human and 94\% of those in mouse overlap annotated mobile elements \cite{Hubley_2015}. Mobile elements are inserted both into intergenic regions and into gene-bodies (Figure 5B), thus contributing to the increase in gene length. 
%The fact that insertions make a greater contribution intergenically than within genes may reflect selection against insertions into exons (ref). 
Taken together, these, and analyses from other groups \citep{Grishkevich_2014,Chimpanzee_2005}, indicate that not only genomes, but also genes are elongated by insertion of specific mobile elements during evolution.

%[need to describe 5C--should this be length of DI/SC, which is just the converse of Fig 4B, or should it be insertions for low noise/hi noise (Fig 5 supp 1A]? Alternatively, we could leave this whole point for the supplement (omit 5C) and explain some of the following logic there: short, low-noise, high SC genes are protected from insertions. This accounts for part of the fact that insertions as a fraction of length actually increase with length. The other portion is the fact mentioned above that insertions into exons are likely selected against.] I favor moving these points to supplement.

Mobile elements (MEs) are known to have contributed to mammalian evolution by carrying regulatory elements \citep{Johnson_2006,Chuong_2016a} and by providing a substrate for the evolution of new regulatory elements through exhaptation \citep{Mikkelsen_2007,Sasaki_2008}. In keeping with a contribution of this process to gene regulation in neurons, ATAC peaks have significant overlap with known repeats (53\%) and this number increases to 93\% when only newly inserted segments are considered (Figure 5 Supplement 1D). To more directly determine if MEs within long genes contribute to shaping their diverse neuronal expression patterns, we asked whether the complement of MEs can be used to predict those expression patterns. We fit gene expression levels with counts of individual MEs within and surrounding each gene (Figure 5C left). The $R^2$ values for each cell type calculated using test genes (20\%) not used for fitting (Figure 5C right, blue) are much larger than expected by chance (Figure 5C right, green/red/orange). If counts and genes are shuffled (green) cross-validated $R^2$ values drop to near 0 (see Figure 5 Supplement 1 for more detail on the different effects of different shuffle controls). 
%([# non-expert question: what does this show? Is it telling me that there isn't some non-related bias or artifact in the data contributing to the correlation?]). 
%We found that if the length of the gene is retained in the shuffling control (orange, red) the $R^2$ values drop to about 1/3 of those in the original fitting. This reflects the fact that gene length is highly correlated with expression
%expression pattern across cells or level with in cell type or both? This part gets a little confusing for me (an "average" reader).] 
%(Figure 4 Supplement 1[#fix supp]; c=0.418: mean Pearsonâ€™s r between log gene length and expression rank). The presence of some repeats, such as SINEs, are highly correlated with both gene length (c=0.841) and expression (Figure 4 Supplement 2[#fix supp]; mean c=0.454) [#suggesting they are key drivers of both elongation and expression diversity?]. 
Although many other factors likely contribute to shaping gene expression across neuronal cell types, consistent with $R^2$ values below 0.3, the regression analysis indicates a highly significant contribution by MEs. These results suggest that the elongation of neuronal effector genes by mobile element insertions have endowed them with increased capacity for differential expression, thereby  enhancing neuronal diversity. 

Our transcriptomic analyses suggest that gene length is an important contributor to gene expression diversity across neuronal cell types in the brain. 
%Cell type diversity is a hallmark of the nervous system, and is thought to be critical for its ability to generate complex behavioral outputs. 
Thus we hypothesized that gene length and the mechanisms driving lengthening over time would contribute to the evolution of nervous system complexity. Interestingly, the expression of long genes is not uniform across brain regions, but is highest in the evolutionary newer forebrain and is lower in the older brainstem and hypothalamus (Figure 5D). 
%Non-neuronal cell types expressed only 1/2 to 1/5 as many long genes as neuronal cell types (blue bars in Figure 5E). This was true even for non-dividing cell types like myocytes and largely non-dividing tissues like the heart (data not shown). [#I feel like this is suggesting some conclusion about neuronal diversity being most critical for brain function diversity which is why they have the most long genes, but I'm not sure how to phrase it. Maybe too redundant with next sentence? Either way, this paragraph feels like it needs an ending.]
The involvement of gene length in cell type diversity predicts that species with longer genes will have a larger, more diverse set of neuronal cell types, permitting construction of more complex neural circuits. Consistent with this, we find that the distribution of gene lengths for the best annotated species is broadly correlated with nervous system size and complexity (Figure 5E). This is also true of neuronal genes, (defined in the mouse), which also increase in length with increased nervous system complexity (data not shown). 

In summary, genes are elongated by mobile element insertions that provide a substrate for new 
%%carry [# the use of carry feels like an over reach just based on our data. Do we know from other work that MEs most often transplant regulatory sites as apposed to evolving into novel ones?] 
regulatory elements which in turn diversify transcription across neuronal cell types. Species endowed with more complex brains have longer sets of genes, consistent with the idea that the elongation of neuronal genes through MEs is linked to expanded neuronal diversity.




