\subsection{Diversity arising from alternative splicing}

Alternative splicing is known to increase transcriptome diversity \citep{ Andreadis_1987}. To assess the contribution of alternative splicing to diversifying transcriptomes across cell populations, we quantified the branch probabilities at each alternative splice donor site within each gene (Figure 6A top). The branch probabilities at each donor site are the relative frequencies with which particular splice acceptors are chosen, and can be estimated from observed junction read counts. Branch probabilities are highly bimodal (Figure 6A bottom), suggesting that most branch point choices are made consistently, in an all-or-none fashion, for any given cell population.

To test the significance of differential splicing across cell populations, we utilized a statistical test based on the Dirichlet-Multinomial distribution and the log-likelihood ratio test, developed in LeafCutter \citep{Li_2016}. We used pair-wise differential expression of each branch to calculate a branch DEF, much as we previously calculated the differentially expressed fraction (DEF) from expression values (Figure 3). Examples of branches with high DEFs are shown in Figure 6B. The list includes known examples like the site of the flip and flop variants of the AMPA receptor subunit Gria2 \cite{Sommer_1990}. Another previously known example is the splicing regulator muscleblind like splicing factor 2 (Mbnl2), which is known to regulate splicing in the developing brain  \cite{Charizanis_2012} and is known to be spliced at multiple sites, including the one shown in Figure 6B \cite{Pascual_2006}. 

In order to determine which families of genes are highly differentially spliced, we computed a splice DEF per gene by combining the ability of a geneâ€™s alternatively spliced sites to distinguish a pair of samples (i.e. a pair is distinguished by a gene if any alternatively spliced site in the gene can distinguish the pair). Using this combined splice DEF, we found that RNA binding proteins, especially splicing related factors (such as Pcbp2 and Mbnl2) are highly alternatively spliced among neuronal cell types\cite{Zheng_2013}, but over-represented categories also included other families such as Glutamate receptors and G-protein modulators (Figure 6C). 

To begin to assess the functional impact of alternative splicing, we determined which alternative sites lead to inclusion or exclusion of a known protein domain using the Pfam database \citep{Finn_2015}. This revealed domain differences between many previously unknown isoforms. For example, Amyloid precursor-like protein 2 (Aplp2), contains a bovine pancreatic trypsin inhibitor (BPTI) Kunitz domain entirely included or excluded by the alternative site preceding it (Figure 6D top). A specific role for this domain in Aplp2 is not known, but the domain is known to regulate proteolysis of the related protein APP, the amyloid precursor protein implicated in Alzheimer's disease \citep{Beckmann_2016}. Isoforms of Aplp2 that include the domain are common in forebrain and non-neuronal cell types, but not in hindbrain neuronal cell types, suggesting its proteolytic processing may differ in these regions.

The analysis also provides a more comprehensive view of known splice events. For example, Kalirin (Kalrn) is a RhoGEF kinase implicated in Huntington's disease, schizophrenia and synaptic plasticity \citep{Penzes_2008}. Kalrn activity is known to be regulated via binding of adaptor proteins to its SH3 (SRC homology 3) domains \citep{Schiller_2006}. An alternative site preceding the first SH3 domain is highly differentially regulated. In most cortical and striatal cell types, Kalrn lacks the SH3 domain (Figure 6D bottom heatmap), thereby eliminating its modulation through this domain. In contrast, the domain is retained in most cell types from the thalamus, hypothalamus and brainstem. Although the importance of this splicing event is known from prior work, here we greatly expand the number of known variants and their distribution across neural cell types. In total, the data reveal a detailed quantitative view of hundreds of thousands of known and unknown cell type-specific splicing events, providing an unmatched resource for investigating their functional significance.

Not all splicing events alter the inclusion or exclusion of known protein domains. Many splicing events introduce frame shifts or new stop codons and hence are predicted to lead to nonsense-mediated decay (NMD). Coupling of regulated splicing to NMD is believed to be an important mechanism for regulating protein abundance \cite{Lewis_2002}. Consistent with previous observations \citep{Yan_2015,Mauger_2017}, we noticed that most alternative sites contain branches that can lead to NMD (Figure 6E). This suggests that alternative splicing may contribute not only to the diversity of transcripts present, but to diversity defined on the basis of transcript abundance. 

The present results provide a comprehensive resource of known and novel splicing events across a large number of neuronal cell types. Altogether, nearly 70\% of alternative sites lead to differential inclusion of a known Pfam domain or NMD (Figure 6E), and thus to functional or quantitative diversity across cell types.
