\subsection{Metrics to Quantify Diversity}
Analysis of expression differences between individual groups is the basis of most profiling efforts. Variance-based metrics, such as Analysis of Variance (ANOVA) F-Value, or coefficient of variation (CV) are commonly used for this purpose. However, these metrics are jointly affected by the pattern of differential expression and the robustness of the differences, and so cannot readily separate these two features (Figure 2A,B; Figure 2 Supplement 1). Since these features may differ in their biological significance, we searched for the simplest way to quantitatively separate them, leading us to adopt two easily calculated metrics that better separate the pattern, or information content, and the robustness of expression differences.

To quantify the contribution of each gene to patterns of cell type diversity, we measured the fraction of cell-type pairs in which the gene is differentially expressed. This differentiation index, is basically identical to the Gini-Simpson's diversity index \cite{Simpson_1949} widely used in ecology (see Figure 2 Supplement 2). The Differentiation Index (DI) defined here is simply the fraction of pairs distinguished; and ranges from 0 to 1. The maximum observed value of 0.65 indicates that the gene distinguishes 65\% of the pairs, while a value of 0 indicates that the gene distinguishes none (i.e., it is expressed at similar levels in all cells). DI is easy to calculate and approximates the mutual information (MI) between expression levels and cell types (Figure 2 Supplement 2). 

The robustness of an expression difference depends on its magnitude relative to the underlying noise. Robustness is often quantified as a Signal-to-Noise-Ratio (SNR). Since the signals we are interested in are the gene expression differences distinguishing cell types, we compared expression differences between distinguished pairs to a noise estimate derived from undistinguished pairs. This estimate of SNR, termed Signal Contrast (SC), indicates the robustness of pair distinctions, but is independent of the number of pairs distinguished. High SC genes robustly distinguish cell populations and are therefore suitable as ”marker genes”. %Example marker genes for each profiled cell type are included in Supplementary table/figure XX.

Unlike DI and SC, traditional variance-based methods like ANOVA F-values and CV are either affected by both MI and SNR (ANOVA; Figure 2B and Figure 2 Supplement 1) or by neither (CV; Figure 2 Supplement 1). The fact that ANOVA does not distinguish between information content and SNR can be appreciated from the fact that high-ANOVA genes (Figure 2B) include both high DI and high SC genes. Therefore, SC and DI are useful because they provide independent measures of the robustness and magnitude of differential expression between cell types.

To determine the types of genes most differentially expressed (highest DI) and most robustly different (highest SC) between cell types, we used the PANTHER gene families (Figure 2C,D). The least noisy expression differences (highest SC) were those of homeobox transcription factors (TFs) and the more inclusive categories (TFs, DNA binding proteins) that include them (Figure 2C). As might be expected, high DI genes are enriched for neuronal effector genes including receptors, ion channels and cell adhesion molecules (Figure 2D). 

Thus, using these two simple metrics we identify synaptic and signaling genes as the most differentially expressed, and homeobox TFs as the most robustly distinguishing families of genes. These two categories of genes drive neuronal diversity by endowing neuronal cell types with specialized signaling and connectivity phenotypes, and by orchestrating cell type-specific patterns of transcription. In addition, their distinct contributions to distinguishing neuronal types suggests possible differences in the regulation of these two categories of genes. 


