\subsection{Long genes contribute disproportionately to neuronal diversity}

We found that neuronal effector genes such as ion channels, receptors and cell adhesion molecules have the greatest ability to distinguish cell types (highest DI; Figure 4A). Previously, these categories of genes have been found to be selectively enriched in neurons and to share the physical characteristic of being long \cite{Sugino_2014}; \cite{Gabel_2015}; \cite{Zylka_2015}. Consistent with this, DI is strongly length biased (Fig.6A). Hence long genes, which are preferentially expressed in neurons (see also Fig.6B), also contribute most to the differential expression between neuronal cell types. 

One likely reason for the differential expression of long genes in neurons is the well known repression of neuronally expressed genes by the zinc-finger transcription factor REST/NRSF in non-neurons \cite{RN1}\cite{RN2a}. To assess the magnitude of this effect and its influence on the length distribution of neuronal genes, we plotted the length-dependence of genes containing RE1/NRSE elements and observed that they are indeed biased toward long genes (Fig.6C). When these REST targets are removed from neuronally expressed genes, the length distribution looks similar to that of non-neurons (Fig.6D) and the length bias of DI flattened(???).

Long genes differ from more compact genes primarily in number and length of their introns, which, for the longest genes, comprise all but a few percent of their length (Figure 7 Supplement). Introns are known to have elongated in vertebrate genomes through insertion of mobile elements, especially retrotransposons (). Mobile elements may also be "domesticated" or "exhapted" as regulatory elements () suggesting the possibility that elongation through insertion of mobile elements may increase the regulatory capacity of long genes, thus augmenting neuronal diversity. In order to determine whether or not the introns of long genes have enhanced regulatory capacity, we identified candidate regulatory elements as sites of enhanced genome accessibility using ATAC-seq on 7 different neuronal cell types (). As expected, long genes had more candidate regulatory elements (ATAC peaks; suppl) and these peaks were present in a greater number of distinct patterns per gene across cell types (Figure 6E). Consistent with a role in differential expression, the number of unique patterns correlated with the degree of differential expression across cell types (Figure 6F,G).

To compare candidate regulatory elements in long genes between neurons and non-neurons, we used publicly available DNase-seq data from the ENCODE project \cite{Dunham_2012}. We found a significantly higher number of open chromatin sites in brain compared to non-brain tissue (Figure 6 supplement 1X, supplemental table). This bias was  particularly pronounced in forebrain, and was stronger in human than in mouse tissue (Figure 6 supplement 1X).