\subsection{Extracting a compact TF code}
In addition to identifying the most informative transcription factors across the entire set of cell types studied, we wished to identify the most informative TFs for individual cell types. To accomplish this, we extracted the most compact set of TFs needed to specify each cell types in terms of whether each TF was “On” or “Off.” The resulting extracted hierarchy of TFs corresponds to a decision tree \cite{Saeys_2007} for efficiently classifying cell types\cite{Gabitto_2016}. At each level of the tree, TFs were chosen to optimimally bisect the set of cell types into two groups that differed maximally from each other in terms of their overall gene expression. To extract a hierarchical set of TFs working at each level of anatomical organization, we favored TFs that distinguished the bisected groups, but not their anatomical subgroups.

The selected TFs included many genes previously implicated as key transcriptional regulators (KTRs) in the development or maintenance of the distinguished cell types. For example, Foxg1, which split forebrain from other cell types, is known to be critically required for normal development of the telencephalon \cite{Xuan_1995}; \cite{Danesin_2012} and is known to function cell autonomously within the olfactory placode for the production of olfactory sensory neurons, as well as for all other cells in the olfactory lineage \cite{Duggan_2008}. (Note that the TF decision tree is based on adult expression, so the retinal ganglion cell (RGC) subtypes sampled are included in the Foxg1- group, even though Foxg1 is known to be expressed in the early eyecup and to contribute to RGC pathfinding \cite{Schulte_2005}). Similarly, at the next levels,  Tbr1\cite{Bedogni_2010}, Satb2\cite{Sweeney_2017}, Egr3\cite{Chandra_2015}, Isl1\cite{Lu_2013} and Emx2 \cite{Zhang_2016}, are known as KTRs involved in the development and/or maintenance of the relevant cell types.

The TF code identified for each cell type is not unique. First, additional TFs are consistent with the tree shown (see Supplementary Table X). Second, the details of the tree depend on the precise procedure used to extract it. We explored variant procedures that better preserved the known anatomical and developmental relationships between cell types (Figure 5 Supplement 1) as well as procedures that made no assumptions about these relationship whatsoever (Figure 5 Supplement 2). Interestingly, in each case the majority of the same genes were identified, suggesting they encode cell type information that is robust to the precise methods used to extract them.

Although the decision tree classifier identifies many known KTRs, it also suggests hypotheses about less studied genes. For example, Tox2 has received little prior study in the CNS, although it has recently been identified and replicated as locus of heritability for Major Depressive Disorder \cite{Zeng_2016}. Based on its position in the tree, we hypothesize that Tox2 is a KTR of midbrain, hypothalamic and hindbrain celltypes. Hence the list of nodal genes represents a rich source of novel hypotheses about transcriptional regulation in genetically identified cell types. Known and hypothesized KTRs identified by the decision tree classifier are tabulated in Supplementary Table X.

The construction of TF code tree utilized the wide sampling of the cell types in the current data set as well as the "intersection method" which is a powerful way to extract genes relevant to a group of cell types. For example, in the above tree case, we prioritized selection of genes whose expression is consistent with anatomy. These shared genes are more likely to be the one which is kept expressed from the last common progenitors. This is the reason why nodal genes are enriched with developmentally important genes.

As another application which utilized wide sampling of cell types and intersection method, we  extracted pan-neuronal genes (Fig.5 Supp.3). These contain well known genes such as Eno2 (Enolase2), which is required for neuronal specific Crebs cycle, Slc2a3 (chrolide transporter) which is necessary in sensing inhibitory transmission, Atp1a3 ().....  Synaptic genes are often differentially expressed among neurons but the genes included in this pan-neuronal list such as Syn1, Stx1b, Stxbp1, Sv2a, Vamp2 signify common components of any neurons pinpointing toward essential parts of the complexes. In this sense, this pan-neuronal gene list represents a list of necessary components of any neurons.

In summary, we've demonstrated how to extract useful (biological) information from the current dataset by constructing a compact TF code in a decision tree form and by extracting pan-neuronal genes, yielding a number of known genes and new candidate genes likely important for the development and maintenance of these cell types.
