\subsection{The homeobox TF family contains the highest information regarding cell types}

SC, like SNR, is a ratio between signal and noise, and so can reflect high expression levels in most ON cell types (high signal), low expression levels in most OFF cell types (low noise), or both. Homeobox genes are not among the most abundantly expressed genes. Their average expression levels (~30 FPKM) are significantly lower than, for example, those of neuropeptides (~90 FPKM). This suggests that the high SC of homeobox TFs depend more on low noise than on their high signal. In fact, most  homeobox TFs have uniformly low expression in OFF cell types (Figure 4A). We quantified this "OFF noise" for all genes and found that homeobox genes are enriched among genes that have both low OFF noise and at least moderate ON expression levels (red dashed region in Figure 4B).

Since tight control of expression may reflect closed chromatin, we measured chromatin accessibility using ATAC-seq (see methods). As expected, compared to high-noise genes (Figure 4C bottom), genes with low OFF noise were more likely to have fewer, smaller peaks within their transcription start site (TSS) and gene body (Figure 4C top, Figure 4D), consistent with the idea that their expression is controlled at the level of chromatin accessibility.

Functionally, the tight control of homeobox TF expression levels may reflect their known importance as determinants of cell identity, and the fact that establishing and maintaining robust differences between cell types may require tight ON/OFF regulation rather than graded regulation. If they are in fact important in cell type determination, their expression pattern should contain high information regarding cell types compared to other families. 

However, as we saw in Figure 3E, homeobox family does not have particularly high DI (mean 0.21) compared to, for example, cyclic nucleotide-gated ion channel (mean DI=0.31, highest) or GABA receptor (0.29, 2nd) and ranked at 16th. We inferred that this is due to the fact that graded expression differences can contribute to DI. Since binary ON/OFF expression pattern may be more important in cell type specification than graded expression patterns, we calculated a binary version of DI (bDI; see methods). With this metric, the homeobox TF family is the most enriched PANTHER family among the top 1000 bDI genes (Figure 4 Supplement 1A) and had the 2nd highest average bDI (0.07) among PANTHER families next to neuropeptide (0.08).

To see more detailed contribution from subfamilies of TFs, we used TF families in HUGO (Figure 4 Supplement 1B). As expected, homeobox subfamilies had the highest average bDIs within various TF subfamilies. Within the homeobox family, the LIM class of homeobox genes, known to be critical for specifying spinal cord cell types [REF Shirasaki], have the highest mean bDI (Figure 4 Supplement 1B). 

Furthermore, to see how well homeobox TF genes separates cell types compared to other PANTHER family genes, we looked at distances between the cell types calculated using each of PANTHER family (Figure 4E). Homeobox TF family has the highest mean between cell type distance and highest z-score compared to random sampling. [ refer to decorrelation? (supp C) ]

Thus, homeobox TFs has the highest average bDI and best family to separates cell types. This suggests that as has been demonstrated for HOX genes in the vertebrate spinal cord and brainstem \cite{Dasen_2009,Philippidou_2013}, and motoneurons \cite{Kratsios_2017} and somatosensory neurons \cite{Zheng_2015} in C. elegans, homeobox TFs, likely in concert with other TFs, encode the transcriptional diversity of cell types more broadly in the mammalian nervous system.
