\subsection{Information-based metrics to quantify differential expression}

Variance-based metrics, such as ANOVA F-Value, or coefficient of variation (CV) are commonly used for analyzing expression differences between individual groups, but they are jointly affected by the pattern of differential expression and the robustness of the differences and so cannot readily separate them. As a complement to these metrics, we developed two easily calculated information-based metrics that better separate the pattern and robustness of expression differences. In order to extract the transcriptional signals related to cell type identity, we quantified each gene's ability to differentiate each pair of profiled cell types. Based on expression levels and variability (Figure 3A; Methods) we compiled a Differentiation Matrix (DM) with elements equal to one or zero depending on whether or not the gene is differentially expressed between each pair of profiles. The Differentiation Index, DI, is simply the fraction of pairs distinguished, excluding self-comparisons; and ranges from 0 to 1. The maximum observed value of 0.65 indicates that the gene distinguishes 65\% of the pairs, while a value of 0 indicates that the gene distinguishes none (i.e., expressed at similar levels in all cells). 

The ability to detect transcriptional differences between cell types depends on both magnitude of these differences and their associated noise. To quantify this, we defined the Signal Contrast (SC), a metric that closely reflects Signal-to-Noise-Ratio (SNR). Since the signals we are interested in are the gene expression differences distinguishing cell types, we used a noise estimate derived from all undistinguished pairs. SC, which indicates how robustly pairs are distinguished, is the ratio of the average effect size for distinguished and undistinguished pairs. High SC genes robustly distinguish cell populations and are therefore suitable as "marker genes".

Our metrics outperform existing metrics such as ANOVA, CV, and Fano factor in distinguishing information content and robustness of differential expression. To illustrate the properties of DI and SC relative to existing metrics such as ANOVA, coefficient of variation (CV) and mutual information (MI), we calculated these metrics against various simulated expression patterns with added noise (Figure 3B). The results (Figure 3B, lower) demonstrate that DI is highly correlated with MI. This makes intuitive sense, since the division of cell types into those that can and cannot be distinguished (DM; Figure 3A) corresponds to a unit of information about cell types provided by a gene expression pattern. As shown below (Methods), in the simplified case of binary expression levels and ignoring noise, DI is an order-preserved transformation of MI. More generally, DI is highly correlated with MI and can be regarded as an approximation to MI (For a related metric see Simpson's index of diversity \cite{SIMPSON_1949}. Thus, DI provides an estimate of the mutual information between expression patterns and cell types contained, that can be rapidly and easily calculated. 

The simulations also show that DI is not well correlated with SNR. For example, high SNR graded patterns correspond to high DI (Figure 3B, lower right), while high SNR specific binary patterns correspond to low DI (Figure 3B, upper left). In contrast, SC is highly correlated to SNR, to which it is closely related. 

Unlike the information-based metrics DI and SC, traditional variance-based methods like Analysis of Variance (ANOVA) F-values and Coefficient of Variation (CV) are either affected by both MI and SNR (ANOVA) or by neither (CV).  These feature differences between metrics are summarized in Figure 2C.

The fact that ANOVA does not distinguish between information content and SNR is also apparent in the data. As shown in Figure 3D, high-ANOVA genes include both high DI and high SC genes. Therefore, SC and DI are useful because they provide independent measures of the robustness and magnitude of differential expression between cell types.