\subsection{Gene Categories Contributing to Neuronal Diversity}

(Fig.4B will be moved to the end of the fig., Fig4A will be split into 4A and 4B, the rest will be renamed accordingly.)

Using newly defined DI and SC, we checked what kind of genes have high DI (highly differentially expressed) or high SC (differentially expressed in a robust way). We used PANTHER categorization for gene family definitions. As expected, high DI genes are enriched with neuronal genes such as receptors, ion channels as well as cell adhesion molecules which specifies connectivity between neurons (Fig.4A). 

On the other hand, high SC genes are highly enriched with homeobox genes (Fig.4B). The high SC can be attained by high expression level on ON samples or low noise level on OFF samples. The average expression level of the homeobox is around 30FPKM and much lower than, for example that of 90FPKM of neuropeptide family. Therefore, we inferred that this high SC of homeobox family is attained by low OFF noise. In fact, this can be readily seen from the expression pattern (Fig.4C). We quantified OFF state noise (defined as standard deviation of samples with FPKM<1) for all genes (Fig.4D) and found that homeobox genes are again particularly enriched (Fig.4E) in genes with low noise but reasonable maximum expression level (red dashed region in Fig.4D ). 

Since tight control of expression is usually associated with a closed chromatin [REF], we checked ATAC-seq data (see method), for non-expressed hoemobox genes (Fig.4F). As expected, compared to noisy genes (Fig.4G), they were more likely to be without peaks (Fig.4H [REPLACE WITH A NEW ONE]), consistent with the idea that their expression is controlled at chromatin level. 

These tight control of expressions of homeobox genes compared to other genes also consistent with the idea that they are the factors that determine the cell types, which suggest their expression pattern must contain high information content regarding cell type. Since we did not see in the top enriched families in high DI genes, we calculated binarized version of DI (bDI), where a pair of cell types are only distinguished by ON or OFF (i.e. disregarding difference in graded expression). As expected, in this metric, homeobox family is at the top of the enriched families (Fig.4I [ADD THIS PANEL BACK]). Furthermore, we calculated mutual information between entire genes of the PANTHER families and found that homeobox family and parent families contain much higher information than expected from random case (Fig.4J)

Thus, by SC analysis, we objectively identified homeobox (and including TF) family as likely determinant of the cell types.  [<= SOMETHING BETTER WAY TO SAY?]



