\subsection{Genes are elongated by TE insertions and TEs carry regulatory information}

The above results indicate that gene length is an important contributor to gene expression diversity across cell types. Gene lengths differ widely across species (Figure 7A and Figure 7 Supplement1A), suggesting genes are elongated during evolution. In fact, evolutionary older genes are longer (Figure 7 Supplement 1B; \cite{Grishkevich_2014}). To better understand mechanisms of gene elongation over mammalian evolution, we examined segments inserted into the human and mouse genomes by comparing them to closely related species (Figure 7B). Plotted in Figure 7B (left) is the length of the inserted segments. Two clear peaks are recognizable in the length histogram of segments inserted into the human genome, corresponding to Alu and L1 repeats. Moreover, around 92\% of the base pairs of the inserted segments overlap with known repeats (Figure 7B inset; \cite{Hubley_2015}). Similar results are observed in the mouse genome (Figure 7B right). These comparisons indicate that genes are elongated by transposable element (TE) insertions. 

Since long genes have a greater number of candidate regulatory elements as indicated by more ATAC-peaks, we asked whether these can originate from mobile elements. As shown in Figure 7C, 56\% of the ATAC peaks overlap known repeats and this number increases to 75\% when only newly inserted segments are considered, indicating that TE may carry regulatory functions. To explore the possibility that TE/repeats contribute to global regulation of neuronal gene expression, we fitted gene expression levels with counts of individual repeats within and surrounding each gene (Figure 7D).  The $R^2$ values for each cell type calculated using test genes (20\%) not used for fitting (Figure 7E, blue) are much larger than expected by chance (Figure 7E, green/red/orange). If counts and genes are shuffled (green) cross validated $R^2$ values drop to near 0. However, if the length of the gene is retained in the shuffling control (orange, red) the $R^2$ values drop to about 1/3 of those in the original fitting. This reflects the fact that gene length is highly correlated with expression (Figure 7 Suplement 1C; c=0.418: mean Pearson's r between log gene length and expression rank) and some repeats, such as SINEs, are highly correlated with both gene length (c=0.841) and expression (Figure 7 Supplement 1C; mean c=0.454). We also varied the size and position of the regions used to count repeats and found that predictions about expression ($R^2$) were best when including the genebody and the adjacent $10\sim 50$kbp. (Figure7 Supplement 1D,E). 

In summary, genes are elongated by insertions of TEs which overlap candidate regulatory elements, and are predictive of relative gene expression levels, suggesting they may increase the capacity of long genes to be differentially expressed. 


