\subsection{Information-based metrics to quantify differential expression}

Variance-based metrics, such as Analysis of Variance (ANOVA) F-Value, or coefficient of variation (CV) are commonly used for analyzing expression differences between individual groups. However, these metrics are jointly affected by the pattern of differential expression and the robustness of the differences and so cannot readily separate the two parameters. As a complement to these metrics, we developed two easily calculated information-based metrics that better separate the pattern and robustness of expression differences. 

First, in order to extract the transcriptional signals related to cell type identity, we quantified each gene's ability to differentiate each pair of profiled cell types. Based on expression levels and variability (Figure 3A; Methods) we compiled a Differentiation Matrix (DM) with elements equal to one or zero depending on whether or not the gene is differentially expressed between each pair of profiles. The Differentiation Index (DI) is simply the fraction of pairs distinguished, excluding self-comparisons; and ranges from 0 to 1. The maximum observed value of 0.65 indicates that the gene distinguishes 65\% of the pairs, while a value of 0 indicates that the gene distinguishes none (i.e., expressed at similar levels in all cells). 

The ability to detect transcriptional differences between cell types depends on both magnitude of difference and associated noise. To quantify this in our second metric, we defined the Signal Contrast (SC), which closely reflects Signal-to-Noise-Ratio (SNR). Since the signals we are interested in are the gene expression differences distinguishing cell types, we used a noise estimate derived from all undistinguished pairs. SC, which indicates how robustly pairs are distinguished, is the ratio of the average effect size for distinguished and undistinguished pairs. High SC genes robustly distinguish cell populations and are therefore suitable as "marker genes".

Our metrics outperform existing metrics such as ANOVA, CV, and Fano factor in distinguishing information content and robustness of differential expression. To illustrate the properties of DI and SC relative to existing metrics, we calculated these metrics against various simulated expression patterns with added noise (Figure 3B). The results (Figure 3B, lower) demonstrate that DI (blue) is highly correlated with mutual information (MI; green). This makes intuitive sense, since the division of cell types into those that can and cannot be distinguished (DM; Figure 3A) corresponds to a unit of information about cell types provided by a gene expression pattern. As shown below (Methods), in the simplified case of binary expression levels and ignoring noise, DI is an order-preserved transformation of MI, although MI is usually significantly more difficult to calculate precisely. More generally, DI is highly correlated with MI and can be regarded as an approximation to MI (For another metric related to DI, see Simpson's index of diversity \cite{SIMPSON_1949}). The simulations also show that DI is fairly independent from SNR. For example, both high and low SNR binary patterns yield same DIs. In contrast, SC (orange) is independent from MI, but is highly correlated to SNR. Thus, DI provides an estimate of the information content of expression patterns on cell types, whereas SC provides an estimate of signal to noise ratio.

Unlike DI and SC, traditional variance-based methods like ANOVA F-values and CV are either affected by both MI and SNR (ANOVA) or by neither (CV). These feature differences between metrics are summarized in Figure 2C. The fact that ANOVA does not distinguish between information content and SNR is also apparent in the data. As shown in Figure 3D, high-ANOVA genes include both high DI and high SC genes. Therefore, SC and DI are useful because they provide independent measures of the robustness and magnitude of differential expression between cell types.

To determine the types of genes most differentially expressed (highest DI) and most robustly different (highest SC) between cell types, we used the PANTHER gene families (Figure 3E). Since the read count (and hence variability) is affected by gene length \cite{Oshlack_2009}, we restricted counts to 1kb from the 3' end of the gene (Figure 3 Supplement 2A). As expected, high DI genes are enriched for neuronal effector genes including receptors, ion channels and cell adhesion molecules (Figure 3E left). This is consistent with the observations that neurons differ most dramatically in their chemical and electrical signaling properties and in their synaptic connectivity. The least noisy expression differences (highest SC) were those of homeobox transcription factors (TFs) and larger categories of TFs and DNA binding proteins which includes homeobox TFs (Figure 3E right). These TFs "drive" expression of effector genes and often important in determination of the cell types. 

In summary, by using newly defined metric, DI and SC, we extracted neuronal effector genes as the most differentially expressed family of genes and homeobox TFs as the most robustly distinguishing family of genes. These two categories of genes are both important in neuronal diversity, one (TFs) determines cell type and drives expression of other genes and the other (neuronal effector genes) determines the phenotypic differences between cell types. 




















