\subsection{Metrics to Quantify Differential Expression}
In order to extract the transcriptional signals related to cell type identity, we quantified each gene's ability to differentiate each pair of profiled cell types. Based on expression levels and variability (Figure 3A; Methods) we compiled a Differentiation Matrix (DM) with elements equal to one or zero depending on whether or not the gene is differentially expressed between each pair of groups. The Differentiation Index, DI, is simply the fraction of pairs distinguished, excluding self-comparisons; and ranges from 0 to 1. The maximum observed value of 0.65 indicates that the gene distinguishes 65% of the pairs, while a value of 0 indicates that the gene distinguishes none (i.e., expressed at similar levels in all cells). 

The ability to detect transcriptional differences between cell types depends not only on the magnitudes of these differences, but also on their associated noise. To quantify this, we defined the Signal Contrast (SC), a metric closely related to the Signal-Noise-Ratio (SNR). Since the signals we are interested in are the gene expression differences distinguishing cell types, we used a noise estimate derived from all undistinguished pairs. Signal Contrast (SC), which indicates how robustly pairs are distinguished, is the ratio of the average effect size for distinguished and undistinguished pairs. 

To illustrate the properties of DI,SC relative to existing metrics such as ANOVA, coefficient of variation (CV) and mutual information (MI), we calculated these metrics against various expression patterns with added noise (Fig~\ref{fig:fig-metrics}B). ANOVA F-values are calculated using cell type groupings. MI is calculated using the contingency table method by quantizing expression levels. 
The results (Fig~\ref{fig:fig-metrics}C) indicate that differentiation indices (DI, sDI) are highly correlated with mutual information (MI, sMI). Mutual information between expression levels and cell types quantitates how much information on cell types is contained in the expression pattern. Therefore, intuitively, the number of distinguishable pairs of cell types (DI) should be highly related to MI. This relationship is explored in more detail in the supplement (Supplement to Figure~\ref{fig:fig-metrics}) and when 1) binary expression level (i.e. only ON/OFF is assessed),2) contribution to MI from expression noise is ignored, and 3) a same discretization scheme is used for both DI and MI, then DI is an order-preserved transformation of MI. In general, DI is highly correlated with MI and can be regarded as an approximation to MI. DI is also highly related to Simpson's index of diversity used in ecology [REF:1949 Nature, Measurement of Diversity]. Thus, in addition to quantifying the amount of differential expression, DI provides an alternative way to calculate the amount of information on cell types contained in the expression pattern. As can be seen from the error bars in Fig~\ref{fig:fig-metrics}C, DI is less susceptible to noise compared to MI. 

ANOVA is the traditional statistics of choice when quantitating differential expression of multiple cell types. Fig~\ref{fig:fig-metrics}C indicates that ANOVA F-values are affected both by information content and signal-to-noise ratio. Therefore, when these aspects need to be separated, ANOVA is not an appropriate metric. For example, ANOVA F-value for Binary high SNR/EP=0.1 is similar to Binary low SNR/EP=0.5 (Fig~\ref{fig:fig-metrics}C), and they are not distinguishable. 

CV is often used to extract highly varying genes when no cell type labeling is available, such as in the case of single cell experiments to classify samples. From Fig~\ref{fig:fig-metrics}C, we can see that CV is insensitive to information content nor SNR, but depends more on EP. Since it is desirable to have high information content as well as high signal-to-noise ratio for classification features, high DI/high SC genes are more appropriate for this purpose than high CV genes. 
% * <schulmanna@janelia.hhmi.org> 2017-06-26T06:10:28.878Z:
% 
% > EP
% are we gonna introduce this metric?
% 
% ^.

Finally, Fig~\ref{fig:fig-metrics}C indicates that signal contrasts (SC,sSC) capture SNR faithfully for binary cases and approximately for graded cases. This is enabled by separation of pairs in two groups using DM (Fig~\ref{fig:fig-metrics}A), which is also used to calculate DI. High SC genes robustly distinguish cell populations, that is, they are suitable for "marker genes". Since sSC has larger dynamic range compared to SC for graded case and less variability for binary cases, and sDI and DI are essentially the same, in the following analyses, we use sDI and sSC instead of DI and SC.

In summary, DI and SC enable to independently measure the amount of differential expression, which can also be regarded as information content of cell types in the expression pattern, and the robustness of differential expression.