\section{Results}

\subsection{A dataset of cell type-specific neuronal transcriptomes}
To begin exploring the diversity in the nervous system, we collected cell type-specific transcriptomes from 2-4 replicates of 168 types of neurons and 15 types of non-neuronal cells (Table 1; Figure 1 Supplement 1). Data from 10 previously published hippocampal cell types \cite{Cembrowski_2016} and two neocortical cell types \cite{Shima_2016} were also included in our analyses . Each neuron type we collected represents a group of fluorescently labeled cells microdissected from a specific region of the mouse brain or other tissue. In most cases, the fluorescent label was genetically expressed in a mouse driver line, but retrograde labeling was used in some cases. Tissue was dissociated and cells isolated based on fluorescence. The pipeline for cell type-specific transcriptome collection is depicted in Figure 1A (see Methods for additional details). Mouse lines were first characterized by generating a high resolution atlas of reporter expression (Figure 1B), then regions containing labeled cells with uniform morphology were chosen for sorting and RNASeq. Manually sorted samples had 132Â±16 cells ($mean\pm sem$, all following as well) and in all cases corresponded to previously described cell types. This effort constitutes the largest and most diverse single collection of cell types profiled by RNA-seq. The processed data, including anatomical atlases, RNASeq coverage, and TPM are available at http://neuroseq.janelia.org (Figure 1C). 

To determine the depth of our transcriptional profiling, we quantified the sensitivity of our library preparation. Amplified RNA libraries had an average sensitivity (50\% detection) of 23 copy*kbs of ERCC spike-ins across all libraries (Figure 1D), indicating our pipeline had the sensitivity to detect a single copy of a transcript per cell 80\% of the time. In total we sequenced 2.23 trillion bp in 558 libraries. Total reads per library was $41\pm0.5M$ reads (Figure 1 Supplement 2A). Using the aligner STAR \cite{Dobin_2012}, $68.9\pm0.37\%$ of the reads mapped uniquely to the mm10 genome, $2.8\pm0.06\%$ mapped to multiple loci, $5.6\pm0.14\%$ did not map to mm10, and $22.7\pm0.36\%$ contained abundant sequences such as ribosomal RNA or mitochondrial sequences (Figure 1 Supplement 2B) and $0.06\%\pm 0.004\%$ contained short reads (less than 30bp after removing adaptor sequences). Sequenced library data were deposited in NCBI GEO (accession\# GSE79238). This high sensitivity allowed for deep transcriptional profiling in our diverse set of cell types.

 To demonstrate an example of the use of the dataset, we extracted pan-neuronal genes, utilizing the wide sampling of the cell types of the dataset (Figure 1 Supplement 3). Wide sampling of the cell types is essential because, if the sampling of the cell types are low, extraction of the pan-neuronal genes can lead to false positives \cite{Mo_2015}\cite{Stefanakis_2015}. Extracted pan-neuronal genes contain well known genes such as Eno2 (Enolase2), which is required for neuronal specific Krebs cycle, Slc2a3 (chrolide transporter) which is necessary in sensing inhibitory transmission, Atp1a3 (ATPase Na+/K+ transporting subunit alpha 3) which belongs to a complex responsible for maintaining electrochemical gradient across the membrane. Synaptic genes are often differentially expressed among neurons but the genes included in this pan-neuronal list such as Syn1, Stx1b, Stxbp1, Sv2a, Vamp2 signify common components of any neurons pinpointing toward essential parts of the complexes. In this sense, this pan-neuronal gene list represents a list of necessary components of any neurons. The dataset should also be useful for many other applications, especially ones which require wide variety of neuronal cell types. 

\begin{table}[p]
%%\begin{fullwidth}
\caption{\label{tab:table1}Summary of the Profiled Samples.}
% Use "S" column identifier to align on decimal point 
%\begin{tabular}{S l l l r}
%%for_pdf%%\resizebox{1\columnwidth}{!}{
\begin{tabular}{l l l l l l}%{ | l | l | l | l | l | l | }
\toprule
	 & region/type & transmitter & \#groups & subregions & \#samples \\ 
\midrule
	CNS neurons & Olfactory (OLF) & glu & 10 & AOBmi,MOBgl,PIR,AOB,COAp & 28 \\ 
	 &  & GABA & 4 & AOBgr,MOBgr,MOBmi & 12 \\ 
	 & Isocortex & glu & 22 & VISp,AI,MOp5,MO,VISp6a,SSp,SSs,ECT,ORBm,RSPv & 52 \\ 
	 &  & GABA & 3 & Isocortex,SSp (Sst+, Pvalb+) & 7 \\ 
	 &  & glu,GABA & 1 & RSPv & 3 \\ 
	 & Subplate (CTXsp) & glu & 1 & CLA & 4 \\ 
	 & Hippocampus (HPF) & glu & 24 & CA1,CA1sp,CA2,CA3,CA3sp,DG,DG-sg,SUBd-sp,IG & 68 \\ 
	 &  & GABA & 4 & CA3,CA,CA1 (Sst+, Pvalb+) & 12 \\ 
	 & Striatum (STR) & GABA & 12 & ACB,OT,CEAm,CEAl,islm,isl,CP & 33 \\ 
	 & Pallidum (PAL) & GABA & 1 & BST & 4 \\ 
	 & Thalamus (TH) & glu & 11 & PVT,CL,AMd,LGd,PCN,AV,VPM,AD & 29 \\ 
	 & Hypothalmus (HY) & glu & 11 & LHA,MM,PVHd,SO,DMHp,PVH,PVHp & 36 \\ 
	 &  & GABA & 4 & ARH,MPN,SCH & 15 \\ 
	 &  & glu,GABA & 2 & SFO & 3 \\ 
	 & Midbrain (MB) & DA & 2 & SNc,VTA & 5 \\ 
	 &  & glu & 2 & SCm,IC & 6 \\ 
	 &  & 5HT & 2 & DR & 10 \\ 
	 &  & GABA & 1 & PAG & 4 \\ 
	 &  & glu,DA & 1 & VTA & 3 \\ 
	 & Pons (P) & glu & 7 & PBl,PG & 22 \\ 
	 &  & NE & 1 & LC & 2 \\ 
	 &  & 5HT & 2 & CSm & 7 \\ 
	 & Medulla (MY) & GABA & 7 & AP,NTS,MV,NTSge,DCO & 16 \\ 
	 &  & glu & 6 & NTSm,IO,ECU,LRNm & 20 \\ 
	 &  & ACh & 2 & DMX,VII & 6 \\ 
	 &  & 5HT & 1 & RPA & 3 \\ 
	 &  & GABA,5HT & 1 & RPA & 4 \\ 
	 &  & glu,GABA & 1 & PRP & 3 \\ 
	 & Cerebellum (CB) & GABA & 10 & CUL4, 5mo,CUL4, 5pu,CUL4, 5gr,PYRpu & 25 \\ 
	 &  & glu & 4 & CUL4, 5gr,NODgr & 10 \\ 
	 & Retina & glu & 5 & ganglion cells (MTN,LGN,SC projecting) & 14 \\ 
	 & Spinal Cord & glu & 1 & Lumbar (L1-L5) dorsal part & 3 \\ 
	 &  & GABA & 4 & Lumbar (L1-L5) dorsal part, central part & 12 \\ 
	PNS & Jugular & glu & 2 & (TrpV1+) & 7 \\ 
	 & Dorsal root ganglion (DRG) & glu & 2 & (TrpV1+, Pvalb+) & 5 \\ 
	 & Olfactory sensory neurons (OE) & glu & 4 & MOE,VNO & 9 \\ 
\midrule     
	non-neuron & Microglia &  & 2 & MOp5(Isocortex),UVU(CB) (Cx3cr1+) & 6 \\ 
	 & Astrocytes &  & 1 & Isocortex (GFAP+) & 4 \\ 
	 & Ependyma &  & 1 & Choroid Plexus & 2 \\ 
	 & Ependyma &  & 2 & Lateral ventricle (Rarres2+) & 6 \\ 
	 & Epithelial  &  & 1 & Blood vessel (Isocortex) (Apod+,Bgn+) & 3 \\ 
	 & Epithelial &  & 1 & olfactory epithelium & 2 \\ 
	 & Progenitor &  & 1 & DG (POMC+) & 3 \\ 
	 & Pituitary &  & 1 & (POMC+) & 3 \\ 
\midrule     
	non brain & Pancreas &  & 2 & Acinar cell, beta cell & 7 \\ 
	 & Myofiber &  & 2 & Extensor digitorum longus muscle & 7 \\ 
	 & BAT &  & 1 & Brown adipose cell from neck.  & 4 \\ 
\midrule  
	 &  & total & 193 &  & 558 \\ 
\bottomrule
\end{tabular}
%%for_pdf%%}
%\end{fullwidth}
\end{table}




