\subsection{Long genes contribute disproportionately to neuronal diversity}

We found that neuronal effector genes such as ion channels, receptors and cell adhesion molecules have the greatest ability to distinguish cell types (highest DI; Figure 3E). Previously, these categories of genes have been found to be selectively enriched in neurons and to share the physical characteristic of being long \cite{Sugino_2014}; \cite{Gabel_2015}; \cite{Zylka_2015}. Consistent with this, DI is strongly correlated with length (Fig.6A). Hence long genes, which are preferentially expressed in neurons (see also Fig.6B), also contribute most to the differential expression between neuronal cell types. 

Long genes are also especially prone to DNA damage in dividing neural progenitors (Wei 2016). More generally, long genes may be more prone to collisions between replication and transcription in dividing cells simply due to their inherent length. Interestingly, the protein REST/NRSF was recently found to protect neural progenitors from DNA damage during neurogenesis \cite{Nechiporuk_2016}. REST is also an important zinc-finger transcription factor restricting expression of neuronal genes in non-neurons \cite{RN1}\cite{RN2a}. We reasoned these observations might be related if REST preferentially targets long genes. To assess the magnitude of this effect and its influence on the length distribution of neuronal genes (Figure 6 Supplement 1A), we plotted the length-dependence of genes containing RE1/NRSE elements (Figure 6 Supplement 1B) and observed that they are indeed biased toward long genes (Fig.6C). When these REST targets are removed from neuronally expressed genes, the length distribution of expressed genes looks similar to that of non-neurons (Figure 6 Supplement 1C). However, consistent with the fact that only 8.6\% of neuronally expressed genes are REST targets (contain an NRSE), the removal of these genes has only a modest effect on the length distribution of DI (Figure 6 Supplement 1D). Therefore, either most  long genes expressed in neurons are also expressed in some non-neuronal cells, or additional mechanisms besides REST must exist to prevent that expression, or both are true. 

The expression of long genes is not uniform across brain regions, but is highest in the forebrain and is lower in the brainstem and hypothalamus (Figure 6 B). Non-neuronal cell types expressed only 1/2 to 1/5 as many long genes as neuronal cell types (blue bars in Figure 6B). This was true even for non-dividing cell types like myocytes and largely non-dividing tissues like the heart (separate data not shown).

Long genes differ from more compact genes primarily in the number and length of their introns, which, for the longest genes, comprise all but a few percent of their length (Figure 6 Supplement). Introns are known to have elongated in vertebrate genomes through insertion of mobile DNA elements, especially retrotransposons (). Mobile elements can be "domesticated" or "exhapted" into regulatory elements () suggesting elongation via insertion of mobile elements may increase the regulatory capacity of long genes. This increase in regulatory capacity could act as a platform on which gene expression can be fine-tuned to facilitate the evolution of neuronal diversity. In order to determine whether or not the introns of long genes have enhanced regulatory capacity, we identified candidate regulatory elements as sites of enhanced genome accessibility using ATAC-seq \cite{Buenrostro_2013} on 7 different neuronal cell types. As expected, long genes had more candidate regulatory elements (ATAC peaks; Figure 6 supplement) and these peaks were present in a greater number of distinct patterns per gene across cell types (Figure 6C-E). Consistent with a role in differential expression, the number of unique patterns correlated with the degree of differential expression across cell types (Figure 6D).

To compare candidate regulatory elements in long genes between neurons and non-neurons, we used publicly available DNase-seq data from the ENCODE project \cite{Dunham_2012}. We found a significantly higher number of open chromatin sites in brain compared to non-brain tissue (Figure 6 supplement , supplemental table). This bias was  particularly pronounced in forebrain, and was stronger in human than in mouse tissue (Figure 6 supplement ). Together these data support the hypothesis that neuronal genes may have increased in length over evolutionary time to support more complex and nuanced regulatory regimes. 