\subsection{Long genes contribute disproportionately to neuronal diversity}

We found that neuronal effector genes, such as ion channels, receptors and cell adhesion molecules have the greatest ability to distinguish cell types (Figure 2D). Previously, these categories of genes have been found to be selectively enriched in neurons and to share the physical characteristic of being long \cite{Sugino_2014}; \cite{Gabel_2015}; \cite{Zylka_2015}. Consistent with this, DI is strongly correlated with length (Figure 4A). We have also analyzed publicly available single cell data and confirmed that this bias is broadly observable (Figure 4 Supplement 1 [#make supp]). 

Long genes ($\gt$100kb) have nearly twice the average ability to distinguish cell types (DI) as shorter genes (Figure 4B), and provide greater separation between cell types, despite being nearly four times fewer in number (Figure 4C). In contrast, the robustness of expression differences between cell types (SC) is not different for long and short genes (Figure 4B).  

Previous work showing that long genes are preferentially expressed in brain was perplexing given the metabolic costs \cite{Castillo_Davis_2002} and impaired genome integrity \cite{Wei_2016} associated with long genes. Our finding that long genes contribute disproportionately to distinguishing neuronal cell types suggest that increased neuronal diversity may be a counterbalancing benefit. We hypothesize that increased length could provide a platform for the evolution of novel regulatory elements, thereby contributing to expression diversity across cell types.

Long genes differ from more compact genes primarily in the number and length of their introns, which, for the longest genes, comprise all but a few percent of their length (Figure 4 Supplement 1). To begin to examine the possibility that longer introns provide a larger number of gene regulatory elements, we examined genome accessibility, since regulatory elements like promoters and enhancers are known to have high accessibility in the cell types in which they are active. We first analyzed ENCODE DNaseI data to identify candidate regulatory regions from peaks of genome accessibility. As expected from their increased expression and increased ability to discriminate neuronal cell types, long genes contained more candidate regulatory regions in brain than in other tissues (Figure 4 Supplement 2B,C)[# fix supp]. 

In order to identify differences in candidate regulatory elements across cell types, we measured sites of enhanced genome accessibility using ATAC-seq \cite{Buenrostro_2013} from 7 different neuronal cell types. As expected, long genes had more candidate regulatory elements (ATAC peaks; Figure 4 supplement 3) and these peaks were present in a greater number of distinct patterns per gene across cell types (Figure 4D-E). Consistent with a role in differential expression, the number of unique patterns correlated with the degree of differential expression across cell types (Figure 4F). Thus, long genes contribute disproportionately to neuronal diversity, likely due to diversity in gene regulation occurring via an increased number of regulatory sites within long genes. 


