\subsection{Genes with the highest information regarding cell types}

To determine the types of genes most differentially expressed (highest DI) and most robustly different (highest SC) between cell types, we used the PANTHER \citep{Thomas_2003} gene families (Figure 3E). As expected, high DI genes are enriched for neuronal effector genes including receptors, ion channels and cell adhesion molecules (Figure 3E top). The least noisy expression differences (highest SC) were those of homeobox transcription factors (TFs) and the more inclusive categories (TFs, DNA binding proteins) that include them (Figure 3E bottom). We have also applied DM/DI method to alternative splicing (see Figure 3 Supplement 1B-D) and found that voltage-gated calcium and sodium channels are highly alternatively spliced consistent with previously known results \citep[e.g.][]{Lipscombe_2013}, but also found G-protein modulators, especially guanyl-nucleotide exchange factors (GEF), are also highly alternatively spliced. 

SC, like SNR, is a ratio between signal and noise, and so can reflect high expression levels in ON cell types (high signal), low expression levels in OFF cell types (low noise), or both. Homeobox genes are not among the most abundantly expressed genes. Their average expression levels (~30 FPKM) are significantly lower than, for example, those of neuropeptides (~90 FPKM). This suggests that the high SC of homeobox TFs depend more on low noise than on their high signal. In fact, most homeobox TFs have uniformly low expression in OFF cell types (e.g. Figure 4A). We quantified this "OFF noise" for all genes and found that homeobox genes are enriched among genes that have both low OFF noise and at least moderate ON expression levels (red dashed region in Figure 4B).

Since tight control of expression may reflect closed chromatin, we measured chromatin accessibility using ATAC-seq \citep{Buenrostro_2013} on 7 different neuronal cell types (see methods). As expected, compared to high-noise genes (Figure 4C bottom), genes with low OFF noise were more likely to have fewer, smaller peaks within their transcription start site (TSS) and gene body (Figure 4C top, Figure 4D), consistent with the idea that their expression is controlled at the level of chromatin accessibility.

Functionally, the tight control of homeobox TF expression levels may reflect their known importance as determinants of cell identity, and the fact that establishing and maintaining robust differences between cell types may require tight ON/OFF regulation rather than graded regulation. If they are, in fact, important "drivers" of cell type-specific differences, their expression pattern should be highly informative about cell types. However, the homeobox family was not identified on the basis of a particularly high DI (Figure 3E and Figure 4 Supplement 1B; mean DI=0.21; rank 16th) compared to, for example, cyclic nucleotide-gated ion channels (mean 0.31, highest) or GABA receptors (0.29, 2nd). We infer that this is due to the fact that graded expression differences also contribute to DI. Since binary ON/OFF expression patterns may be more critical for cell type specification than graded expression patterns, we calculated a binary version of DI (bDI; see methods). With this metric, the homeobox TF family is the most enriched PANTHER family among the top 1000 bDI genes (Figure 4 Supplement 1A) and had the 2nd highest average bDI (0.07) among PANTHER families after neuropeptides (0.08) (Figure 4 Supplement 1B). Among TF subfamilies, the LIM domain subfamily of homeobox genes had the highest mean bDI (Figure 4 Supplement 1C), consistent with its known role in specifying spinal cord and brainstem cell types \citep{Dasen_2009,Philippidou_2013}. 

The ability of gene families to provide information about cell types is determined by both how informative individual family members are, and the relationships between them. If the information across family members is independent, the overall information is increased relative to the case in which multiple members contain redundant information (Figure 4 Supplement 1D). This aspect of "family-wise" information is not captured by "gene-wise" metrics like mean bDI, or by enrichment analysis (Figure 3E, Figure 4 Supplement 1A-C). One way of capturing the additive, non-redundant information within a gene family is to measure its ability to separate cell types using a distance metric. This analysis (Figure 4E) reveals that homeobox TFs have the highest separability. Thus, homeobox TFs provide the best separation of profiled cell types both individually (Figure 4 supplement 1A,B) and as a family (Figure 4E). It has long been known that a subset of homeobox TFs, the HOX genes, play an evolutionarily conserved role in specifying cell types in invertebrates \citep{Kratsios_2017,Zheng_2015} and in the vertebrate spinal cord and brainstem \citep{Dasen_2009,Philippidou_2013}. These analyses suggest that the larger family of homeobox TFs play a broader role in transcriptional diversity of cell types across the mammalian nervous system.

In summary, by defining novel metrics DI and SC, we identify homeobox TFs as the most robustly distinguishing families of genes, synaptic and signaling genes as the most differentially expressed genes. These two categories of genes drive neuronal diversity by orchestrating cell type-specific patterns of transcription and by endowing neuronal cell types with specialized signaling and connectivity phenotypes.



