\subsection{Long genes contribute disproportionately to neuronal diversity}

Neuronal effector genes such as ion channels, receptors and cell adhesion molecules are known to be long genes that are enriched in neurons and frequently linked to CNS developmental disorders (Gabel et al., 2015; Sugino et al., 2014; Wei et al., 2016; Zylka et al., 2015). We found that these gene categories have the greatest ability to distinguish cell types (highest DI; Figure 4A). Consistent with this, DI is strongly length biased (Fig.6A) and neurons express more long genes compared to non-neurons (Fig.6B). Hence long genes are both preferentially expressed in neurons and contribute most to the differential expression between neuronal cell types. 

One likely reason for the differential expression of long genes in neurons is the well known repression of neuronally expressed genes by REST/NRSF in non-neurons (). To assess the magnitude of this effect and its influence on the length distribution of neuronal genes, we plotted the length-dependence of genes containing NRSE elements and observed that they are indeed biased toward long genes (Fig.6C). When these REST targets are removed from neuronally expressed genes, the length distribution looks similar to that of non-neurons (Fig.6D) and the length bias of DI flattened(???).

Long genes differ from more compact genes in the number and length of their introns, which make up as much as 99\% of their length (Figure 6 Supplement 1X?) These introns are known to harbor a diversity of regulatory sites that could contribute to their ability to be differentially expression across a larger number of neuronal cell types. To identify candidate regulatory elements potentially contributing to cell-type specific gene expression, we measured genome accessibility in 7 cell types using ATAC-seq. As expected, long genes had more candidate regulatory elements (ATAC peaks; suppl) and these peaks were present in a greater number of distinct patterns per gene across cell types (Fig. 6E). Consistent with a role in differential expression, the number of unique patterns correlated with the degree of differential expression across cell types as assessed by DI and ANOVA (fig. 6F include ANOVA).

To compare candidate regulatory elements in long genes between neurons and non-neurons, we used publicly available DNase-seq data from the ENCODE project (ref). We found a significantly higher number of open chromatin sites in brain compared to non-brain tissue (sup.fig., sup.table). This bias was  particularly pronounced in forebrain, and was even stronger in human tissue compared to mouse tissue (sub 6C).