\subsection{Gene Categories Contributing to Neuronal Diversity}

(Fig.4B will be moved to the end of the fig., Fig4A will be split into 4A and 4B, the rest will be renamed accordingly.)

Using newly defined DI and SC, we checked what kind of genes have high DI (highly differentially expressed) or high SC (differentially expressed in a robust way). We used PANTHER categorization for gene family definitions. For this purpose, we used whole gene expression level estimated from read counts on genes. Since replicate variance for the whole gene count is affected by gene length due to differences in total counts [REF Oshlack], we used counts up to 1000bp from 3' end of the gene (Fig4.Supp.1A). For differential expression criteria, we used q-values calculated from limma voom (th=0.05) and log fold-change threshold of 2.

As expected, high DI genes are enriched for neuronal genes such as receptors, ion channels as well as cell adhesion molecules which specify connectivity between neurons (Fig.4A). This is consistent with the observations that neurons mainly differ in their input and output properties and in their connectivities.

On the other hand, high SC genes are highly enriched with homeobox genes (Fig.4B). The high SC can be attained by uniformly high expression level in ON cell types or low noise level on OFF cell types. The average expression level of homeobox genes is around 30 FPKM and much lower than, for example that of ~90 FPKM of neuropeptide family. Therefore, we inferred that this high SC of the homeobox family is attained by low OFF noise. In fact, this can be readily seen from the expression pattern of homeobox genes (Fig.4C). We quantified OFF state noise (defined as a standard deviation of samples with $FPKM<1$) for all genes (Fig.4D) and found that homeobox genes are again particularly enriched (Fig.4E) in genes with low noise and reasonable maximum expression level (red dashed region in Fig.4D ).

Since tight control of expression is usually associated with a closed chromatin [REF], we checked ATAC-seq data (see method), for OFF hoemobox genes (Fig.4F). As expected, compared to noisy genes (Fig.4G), they were more likely to be without peaks (Fig.4H [REPLACE WITH A NEW ONE]), consistent with the idea that their expression is controlled at the chromatin level.

The tight control of expressions of homeobox genes compared to other genes is also consistent with the idea that they are the factors that determine the cell identity, which suggests their expression pattern must contain high information content regarding cell type. Since we did not see homeobox in the top enriched families in high DI genes, we calculated a binarized version of DI (bDI), where a pair of cell types are only distinguished by ON or OFF states (i.e. disregarding difference in graded expression). As expected, in this metric, homeobox family is at the top of the enriched families (Fig.4I [ADD THIS PANEL BACK]). Within the homeobox family, LIM class homeobox genes, which are known to be critical in specifying the spinal cord cell types [REF Shirasaki], have the highest mean bDI (Fig.4 Supp.1B). Furthermore, we calculated mutual information between the cell types and the entire genes of the PANTHER families and found that homeobox family and parent TF families contain much higher information than expected from random (Fig.4J)

Thus, using SC analysis, we objectively identified homeobox family as well as parent TF families as likely determinants of the cell types. [BETTER WAY TO SAY THIS? CONNECTION LOGIC TO THE NEXT SECTION?]
