\subsection{Long genes contribute disproportionately to neuronal diversity}

We found that neuronal effector genes (ion channels, receptors and cell adhesion molecules) have the greatest ability to distinguish cell types (Figure 2D). Previously, these categories of genes have been found to be selectively enriched in neurons and to share the physical characteristic of being long \cite{Sugino_2014}; \cite{Gabel_2015}; \cite{Zylka_2015}. Consistent with this, DI is strongly correlated with length (Figure 4A). We have also analyzed publicly available single cell data and confirmed that this bias is observable universally (Figure 4 Supplement 1 [#make supp]). 

Long genes ($\gt$100kb) have nearly twice the average ability to distinguish cell types (DI) as shorter genes (Figure 4B), and provide greater separation between cell types, despite being nearly four times fewer in number (Figure 4C). In contrast, the robustness of expression differences between cell types (SC) is not different for long and short genes (Figure 4B). [#describe Figure 4C?] 

Previous work showing that long genes are preferentially expressed in brain was perplexing given the costs associated with long genes [#refs]. Our finding that long genes contribute disproportionately to distinguishing neuronal cell types suggested this differentation may be a counterbalancig benefit. We hypotheszied that increased length could contribute to neuronal diversity through diversity in gene regulation and that the increase in sequence within these genes could provide a platform for the evolution of novel regulatory elements.

Increased differential expression in long genes may be caused by increased number of regulatory sites available in longer introns. To see this is the case, we first analyzed ENCODE DNaseI data comparing brain vs. other tissues. As expected from their increased expression and increased ability to discriminate cell types in the nervous system, analysis of ENCODE data revealed that candidate regulatory peaks in long genes are more abundant in tissue level genome accessibility data from the nervous system than from outside of the nervous system (Figure 4 Supplement 1B,C)[# fix supp]. This difference is also greater in human tissues than in mouse tissues (Figure 4 Supplement 1D,E)[#fix supp]. [### remove this section? ENCODE data is not that impressive ###]

In order to determine differences of candidate regulatory elements between cell types, we identified sites of enhanced genome accessibility using ATAC-seq \cite{Buenrostro_2013} from 7 different neuronal cell types. As expected, long genes had more candidate regulatory elements (ATAC peaks; Figure 4 supplement 2) and these peaks were present in a greater number of distinct patterns per gene across cell types (Figure 4D-E). Consistent with a role in differential expression, the number of unique patterns correlated with the degree of differential expression across cell types (Figure 4F). Thus, long genes contribute disproportionately to neuronal diversity, likely due to diversity in gene regulation occuring via an increased number of regulatory sites within long genes. 


