\textbf{(A)} Number (histograms) and ratios (dots) of genes expressed in neurons (pink histogram, red dots) and non-neurons (brown histogram, green dots) relative to expression in the entire population (grey histogram) as a function of gene length (ratios computed per bin of 500 genes). 
\textbf{(B)} Number (green histogram) and ratios (green dots) of genes with nearby NRSE relative to the number of neuronally expressed genes (pink histogram). 
\textbf{(C)} (Magenta dots) ratio of neuronally expressed non-REST target genes to the population.
\textbf{(D)} Variation in expression of long genes (as a percentage of all genes expressed) for different brain regions and cell types. Data are assessed separately for sorted cell samples (Left) and publicly available tissue samples (Right; see Methods.
\textbf{(E)} PANTHER gene families enriched among long and short neuronal genes. 
\textbf{(F)} Genomic regions occupied by long and short genes. Although long genes are fewer in number they occupy significantly more of the genome. Long \& short: regions in which long and short genes overlap. 
\textbf{(G)- (K)} Length dependence of peak counts in genome accessibility data.
Examples from mouse encode analyses of genome accessibility in forebrain \textbf{(G)} and liver \textbf{(H)} tissue samples showing individual peaks (black dots) and binned averages (red dots) as a function of gene length. Average mouse \textbf{(I)} and human \textbf{(J)} genome accessibility from brain and non-brain samples.
\textbf{(K)} Peak number vs. gene length for ATAC-seq measurements of genome accessibility in sorted neuronal cell types
\textbf{(L)} Fraction of gene length attributable to intron length.