%%width=0.8\columnwidth
\textbf{Supplementary to Figure 7. TE insertions elongate genes and contain information about gene expression}
\textbf{(A)} Example of gene length differences between species for \textit{Kcnma1} (a calcium-activated potassium channel gene called \textit{slopoke}, in \textit{Drosophila}). 
\textbf{(B)} Estimated evolutionary age of human genes correlates with their length. The length distribution of human genes is plotted as a function of age, estimated from their most distant homologs. Genes common to all vertebrates (or to all listed genomes) are longer than genes common only to mammals (mouse) or common only to primates (monkey).
\textbf{(C)} Correlation between gene expression rank and gene length (blue) and SINE repeat score (orange) calculated for all cell types. Because of how common they are, SINE repeat scores are correlated with gene length.
\textbf{(D)} Similar to Figure 7D but using repeat scores calculated from different sized intervals surrounding each gene (not including genebody). Average $R^2$ is maximal near 10kb for both upstream and downstream. Shuffling conditions are colored as in Figure 7D.
\textbf{(E)} Similar to Figure 7D but for repeat scores calculated from genebody only (upper panel) or genebody$+/-$100kbp (lower panel).
\textbf{(F)} Fraction of genome spanned by long genes (orange) is greater than that by short genes (green), despite being fewer in number. Some genomic regions contain overlapping long and short genes (yellow).
\textbf{(G)} Percentage of inserted sequences calculated in Figure 7A (Human vs. Chimp and Mouse vs. Rat), that overlap TEs within long ($\geq$ 100kbp) or short ($<$100kbp) genes. 