%%width=0.8\columnwidth
\textbf{Long genes have a greater capacity for differential expression.}
\textbf{(A)} Black dots: DI of each gene is plotted against sorted gene length. Red dots: binned average of DIs (1000 genes per bin, sorted by length). Cor.coef. between DI and gene length is 0.19. 
\textbf{(B)} Fraction of expressed genes within the longest 500 genes. Neuronal (red bars) and non-neuronal cell types (blue bars) within each brain region profiled.
\textbf{(C)} (Black dots) Number of distinct patterns of peaks observed in 7 ATAC-seq profiled cell types plotted against the gene length for each gene. Since there are only 7 cell types, there are at most 7 different patterns of peaks (7 being unique pattern for each cell type). (Red dots) Binned average as in panel A. (Background histogram) histogram of number of genes. 
\textbf{(D)} Violin plot showing the relationship between DI and the number of different patterns of ATAC-seq peaks. Cor.coef (0.31) is bigger than that of between DI and gene length (0.19).
\textbf{(E)} ATAC-seq peaks for \textit{Gabra5} showing different patterns of peaks for each cell type. Scale is shown at top right (RPM;read per million).  Expression levels for each cell type are shown at right (gray bars).
\textbf{(F)} Average metrics for long ($\geq$100kbp) and short ($<$100kbp) neuronal genes (reproducibly expressed in neuronal cell types). 
\textbf{(G)} Similar to Figure 4E. Mean distance of pairs of cell types calculated using long neuronal genes and short neuronal genes. Z-score is 33.2 for long and 22.1 for short neuronal genes. 1-Pearson's cor. coef. is used for distance.
